\documentclass[9pt,twocolumn,twoside]{rilabRxiv}
% Use the documentclass option 'lineno' to view line numbers
\setlength{\marginparwidth}{2cm}
\usepackage[textsize=tiny,colorinlistoftodos]{todonotes} % comments in margins
\definecolor{cornflowerblue}{rgb}{0.39, 0.58, 0.93}
\usepackage{blindtext}


%%%%%%%Add comments in color
\newcommand{\ms}[1]{{\small \textcolor{green}{#1}}}
\newcommand{\jri}[1]{{\small \textcolor{red}{#1}}}
\newcommand{\citex}[1]{{\small \textcolor{red}{CITE(#1)}}}
\newcommand{\X}{{\textcolor{red}{X}}}
\newcommand{\mex}{\textit{mexicana}\xspace}
\newcommand{\invfour}{\textit{Inv4m}\xspace}
\newcommand{\fdrgt} {$\textrm{\textit{FDR}} > 0.05$}
\newcommand{\fdreq} {$\textrm{\textit{FDR}} = 0.05$}
\newcommand{\fdrls} {$\textrm{\textit{FDR}} < 0.05$}
\newcommand{\parv}{\textit{parviglumis}\xspace}
\newcommand{\jmjii}{\textit{jmj2}\xspace}
\newcommand{\jmjiv}{\textit{jmj4}\xspace}
\newcommand{\jmjvi}{\textit{jmj6}\xspace}
\newcommand{\jmjix}{\textit{jmj9}\xspace}
\newcommand{\arabidopis}{\textit{Arabidopsis}\xspace}

\newcolumntype{b}{X}
\newcolumntype{s}{>{\hsize=.5\hsize}X}

% Set supplement numbers to S and start counting newly
\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }


\usepackage{hyperref}
\usepackage{CJKutf8}
% \begin{CJK}{UTF8}{min}
% \verb|¯\_(ツ)_/¯|
% \end{CJK}

\title{Multi-Omics of Maize $\textit{Inv4m}$ in Phosphorus Restriction Show Typical Starvation Responses and Leaf-Age Dependency, Rather Than Adaptive Contributions from the Chromosomal Inversion.}
%\title{Does maize \textit{Inv4m} contribute to adaptation to soils with low phosphorus? Not really}

\author[$1$,$2$,*]{Fausto Rodríguez-Zapata}
\author[$1$,$2$]{Nirwan Tandukar}
\author[$1$]{Ruthie Stokes}
\author[$3$]{Allison Barnes}
\author[$4$]{Sergio Pérez-Limón}
\author[$4$]{Melanie Perryman}
\author[$5$]{Miguel A. Piñeros}
\author[$6$]{Daniel Runcie}
\author[$4$]{Ruairidh Sawers}
\author[$1$,*]{Rubén Rellán-Álvarez}

\affil[$1$,*]{Department of Molecular and Structural Biochemistry, N.C. Plant Sciences Initiative, North Carolina State University, Raleigh, NC, USA.}
\affil[$2$]{Genetics and Genomics Program, North Carolina State University, Raleigh, NC, USA}
\affil[$3$]{United States Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, NC 27695}
\affil[[$4$]{Department of Plant Science, Pennsylvania State University, University Park, PA, USA}
\affil[[$5$]{Robert W. Holley Center for Agriculture and Health, USDA-ARS, Ithaca, NY, USA}
\affil[$6$]{Department of Plant Sciences, University of California, Davis, CA, USA}


\keywords{Local Adaptation, highland maize, Teosinte Mexicana introgression}

\runningtitle{The Role of \textit{Invm4} in adaptation to low phosphorus availability} % For use in the footer

%% For the footnote.
%% Give the last name of the first author if only one author;
\runningauthor{Rodríguez-Zapata}
%% last names of both authors if there are two authors;
% \runningauthor{FirstAuthorLastname and SecondAuthorLastname}
%% last name of the first author followed by et al, if more than two authors.
\runningauthor{Rodríguez-Zapata \textit{et al.}}


%%% Abstract %%%%%%%%%%%%%%%%%%
\begin{abstract}
Local adaptation of a species involves the selection of adaptive alleles that confer a fitness advantage in their local environment. Inversions prevent recombination between the standard and inverted heterozygous hybrids. Inversions can play a crucial role in local adaptation by locking together a set of co-adapted alleles, referred to as supergenes. 
\textit{Inv4m} is a 13 Mb inversion in maize that is particularly prevalent in highland maize and highland maize relatives from Mexico. Maize from the highlands of the Trans-Mexican volcanic belt has been shown to be well-adapted to the volcanic, acidic soils with low phosphorus availability. \textit{Inv4m} carries several genes involved in P acquisition and utilization. We therefore hypothesized that \textit{Inv4m} may be involved in adaptation to low phosphorus levels. To test this hypothesis, we introgressed a highland maize variety from the highlands of Michoacan in México into the temperate line B73 and developed Near Introgresion Lines carrying \textit{Inv4m}. We then grew NILS carrying the inversion and controls in soils with low and high phosphorus and evaluated fitness effects of the inversion as well as changes in gene expression using RNA-Seq.  
Here, we demonstrate that P starvation elicits highly conserved transcriptomic, lipidomic, and ionomic responses across near-isogenic lines that differ at the \invfour inversion. Thousands of genes, multiple lipid classes, and major nutrient pools respond strongly to P availability. However, we did not observe significant interactions with \textit{Inv4m}. Nonetheless, a small number of genotype-by-phosphorus interactions exceed statistical thresholds, suggesting secondary modulation of conserved responses. These exceptions may be linked to phenotypic variation in height and flowering, which is dependent on the \invfour karyotype, thereby connecting nutrient stress to developmental control. Our results highlight the robustness of P starvation responses and provide an entry point for dissecting outlier interactions of potential adaptive significance. 
\end{abstract}
%%%%%%%%%%%%%%%%%%%%%%%%%%


\setboolean{displaycopyright}{true}

\begin{document}

\maketitle
\thispagestyle{firststyle}
%\firstpagefootnote
\correspondingauthoraffiliation{
Department of Molecular and Structural Biochemistry, N.C. Plant Sciences Initiative, North Carolina State University, Raleigh, NC, USA.
E-mail: frodrig4@ncsu.edu, rrellan@ncsu.edu}
\vspace{-11pt}%

\setboolean{displaylineno}{true}
\ifthenelse{\boolean{displaylineno}}{\linenumbers}{}

\section{Introduction}

\lettrine[lines=2]{\color{color2}M}aize might have preadapted to temperate zones through introgression from its highland wild relative, \textit{Zea mays ssp. mexicana} (shorthand \mex) \cite{yang2023}. 
Maize was originally domesticated in the tropical lowlands of Mexico. Before its expansion into temperate regions, maize was introduced to the Mexican highlands and the Southwestern United States, where sympatry with highland teosinte likely facilitated the introgression of adaptive alleles from \mex.

However, not all highland-adaptive loci are present in temperate maize. Highland-associated chromosomal inversions, such as \invfour and \textit{Inv9f}, are prevalent in highland teosinte populations \cite{pyhajarvi2013} and traditional Mexican maize varieties (TVs) \cite{crow2020,gonzalez-segovia2019-jy} but are rare in temperate maize. 
Chromosomal inversions can contribute to local adaptation by preserving locally adapted alleles across multiple loci and reducing recombination within the inversion \cite{kirkpatrick2006b}. 
Genotyping of teosinte populations using the Maize 50K chip revealed that \invfour spans 13 Mb and is predominantly found in \mex populations \cite{pyhajarvi2013}. 
In Mexican TVs, variation in \invfour is associated with elevation and flowering time \cite{romero_navarro2017-cn}. 
Additionally, \invfour shows reduced genetic diversity, a clinal relationship with elevation, and is nearly fixed in locations above 2500 m.a.s.l. \cite{crow2020}. 
The inversion exhibits suppressed recombination, as confirmed in a biparental cross \cite{gonzalez-segovia2019-jy}.
\invfour demonstrates classic patterns of gene-by-environment interactions indicative of local adaptation. 
Plants carrying the \invfour-highland allele exhibit delayed flowering at low elevations and earlier flowering at high elevations \cite{crow2020}. 
The highland haplotype of \invfour was introgressed from \textit{Zea mays ssp. mexicana} \cite{pyhajarvi2013,calfee2021-mr,hufford2013-gs}, a wild maize relative native to the Mexican highlands.

Despite strong evidence linking \invfour to local adaptation, the physiological processes and environmental factors underlying its adaptive role remain unclear. 
Furthermore, the specific genes within \invfour that drive local adaptation are largely unidentified. 
Previous research has shown that \invfour-highland upregulates photosynthesis genes in response to cold at the seedling stage \cite{crow2020} and is associated with earlier flowering in the Mexican highlands, which likely enhances fitness in environments with limited growth-degree accumulation throughout the year \cite{romero_navarro2017-cn}. 
However, cold is not the only limiting factor for plant growth in the highlands.
Volcanic soils (Andosols), which dominate the Mexican highlands, present an additional constraint. 
Approximately 95\% of natural Andosol profiles in Mexico are found above 2000 m.a.s.l. \cite{paz-pellat2018,inegi2013}. 
These soils are characterized by high phosphorus retention \cite{krasilnikov2013}, which leads to low phosphorus availability for plant uptake \cite{galvan-tejada2014}. 
MICH21, one of the Mexican highland maize accessions analyzed by \cite{crow2020}, originates from the Purépecha Plateau, where Andosols and phosphorus-efficient TVs are common \cite{paz-pellat2018,galvan-tejada2014,bayuelo-jimenez2011,bayuelo-jimenez2014}.
\invfour may contribute to adaptation in the highlands by carrying alleles that enhance the phosphorus starvation response (PSR). 
For example, the phosphate transporter gene \textit{ZmPho1;2a}, located within \invfour, is a strong candidate for adaptation to low phosphorus availability \cite{salazar-vidal2016-rl}.

In this study, we aimed to understand the physiological and molecular effects of \invfour and to identify candidate genes within the inversion that could elucidate its adaptive role. 
Specifically, we tested whether \invfour-highland contributes to adaptation to low phosphorus availability. To achieve this, we backcrossed MICH21, a Mexican highland TV carrying \invfour, into the B73 genetic background for eight generations, generating Near Isogenic Lines (NILs) \cite{crow2020}. 
These NILs were grown under temperate field conditions with two phosphorus treatments to evaluate flowering time, height, and transcriptomic responses.

We observed that \invfour significantly reduces flowering time and increases plant height, independent of phosphorus levels. We identified a cluster of JUMONJI methyltransferases, which have higher copy numbers in modern maize compared to highland teosinte and TVs, as potential contributors to flowering time differences. Additionally, we found a group of cell cycle-related genes that may underlie height differences. 
While classical PSR genes were identified within the inversion, \invfour had no detectable effect on the phosphorus starvation response. 
These findings provide insights into the genetic mechanisms underlying \invfour’s contribution to local adaptation and highlight potential candidate genes driving its effects.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}


\subsection*{Phosphorus Starvation Dominates Maize Phenotypic Effects with \textit{Inv4m} Dependency Restricted to Cob Diameter.}

Phosphorus deficiency delayed flowering, reduced biomass, and diminished yield  (Fig.~\ref{fig:phenotypes}) in both control (CTRL) and  \textit{Inv4m} lines.
Under $-P$ conditions, anthesis and silking occurred more than three days later relative to $+P$ (Fig.~\ref{fig:phenotypes}~A and B; anthesis: $3.60 \pm 0.26$~days, $p = 2.3 \times 10^{-20}$; silking: $3.42 \pm 0.23$~days, $p = 5.7 \times 10^{-22}$; marginal effect estimate $\pm$ S.E, \textit{FDR} adjusted \textit{p-value}).
In reduced phosphorus, the 50-kernel weight decreased by nearly $18\%$ ($-1.86 \pm 0.37$~g, $p = 8.2 \times 10^{-6}$, Fig.~\ref{fig:phenotypes}~C).
Biomass accumulation was diminished under phosphorus starvation at all measured time points (Fig.~\ref{fig:phenotypes}~E): stover dry weight declined by $5.09 \pm 0.71$~g at 40~DAP ($p = 1.3 \times 10^{-9}$), $16.45 \pm 1.11$~g at 50~DAP ($p = 1.8 \times 10^{-20}$), and $27.94 \pm 3.47$~g at 60~DAP ($p = 1.4 \times 10^{-10}$).
By harvest time (121~DAP), stover biomass remained around $18.5\%$ lower ($-18.70 \pm 3.27$~g, $p = 3.9 \times 10^{-7}$).
Fitted logistic growth curves captured the effect of P-starvation on multiple model parameters (Fig.~\ref{fig:growth}). -P treatment significantly reduced the area under both the empirical (AUCE; $-1.96 \pm 0.18~\text{kg} \times \text{day}$, $p=2.2 \times 10^{-14}$) and logistic growth curves (AUCL; $-1.80 \pm 0.19~\text{kg} \times \text{day}$, $p=2.0 \times 10^{-12}$). Additionally, it reduced the fitted maximum stover weight ($\text{STW}_{\text{max}}$) by approximately $23.00 \pm 3.26$ g ($p = 5.6 \times 10^{-9}$) and delayed the time to reach half maximum stover weight ($\text{T}_{1/2}$) by $3.49 \pm 0.80$ days ($p = 6.0 \times 10^{-5}$), relative to the $+\text{P}$ treatment. We found no significant difference in the rate of stover biomass accumulation.
These phenotypic changes match the canonical maize phosphorus starvation syndrome, indicating a robust physiological response to nutrient limitation.
Crucially, no significant genotype-by-phosphorus interactions were detected for the primary agronomic traits, implying that the \textit{Inv4m} inversion did not alter the direction or magnitude of the main phosphorus response.
Nonetheless, we observe a significant $G \times E$ interaction effect for cob diameter, a secondary reproductive trait.
Specifically, while the cob diameter of control lines was unaffected by phosphorus starvation ($0.19 \pm 0.70$ cm, $p= 0.79$), the $\textit{Inv4m}$ plants grew a cob  $10.7\%$ thinner under nutrient limitation ($-2.81 \pm 0.68$ cm, $p= 1.4 \times 10^{-4}$; conditional effect estimate $\pm$ S.E, \textit{FDR} adjusted \textit{p-value} Fig.~\ref{fig:phenotypes}~D).
Aside from cob diameter, the effects of $\textit{Inv4m}$ were independent of phosphorus conditions and smaller than those of phosphorus starvation.

In both +P and -P treatments, the \textit{Inv4m} plants flowered earlier (anthesis: $-1.31 \pm 0.26$~days, $p = 4.6 \times 10^{-6}$; silking: $-0.93 \pm 0.23$~days, $p = 1.3 \times 10^{-4}$), grew taller by $6.41 \pm 1.05$~cm ($p = 7.7 \times 10^{-8}$), and accumulated less stover biomass at harvest ($-7.94 \pm 3.27$~g, $p = 1.8 \times 10^{-2}$; Fig. Supplementary figure).
% Phosphorus deficiency, on average, caused larger relative reduction of stover biomass than the effect due to \textit{Inv4m} throughout the measured time span (ANCOVA main effect $\beta = 54  \pm 13 \%$, $p = 0.0152$.
%Fig.~\ref{fig:phenotypes}~F)
% We found a significant linear time dependence for -P relative reduction in biomass ($p = 0.027$) but the time effect was not significant for the inversion ($p = 0.90$).The relative reduction of biomass explained by P starvation decreased as plants matured, from $48\%$ at 40~DAP to $18\%$ at harvest.
Overall, while phosphorus starvation consistently resulted in severe developmental and yield penalties whike the $\textit{Inv4m}$ inversion had a smaller and mostly independent influence on the plant phenotype.


\begin{figure*}[!ht]
\centering
\includegraphics[width=0.87\textwidth]{figs/phenotypes.png}
\caption{
\textbf{Maize Response to Phosphorus Starvation is Largely Additive, with $\textit{Inv4m}$ Modulating Cob Diameter.}\\
\textbf{(A)} Experimental design showing the four sampled leaves from $\textit{Inv4m}$ and control (CTRL) NILs.
An increasing number corresponds to older leaves.
\textbf{(B)} Aerial view of the experimental field at Rocksprings, PA.
Boxplots show phenotypic responses of control (CTRL) and \textit{Inv4m} genotypes under phosphorus sufficiency (+P) and deficiency (-P). 
Phosphorus starvation led to delayed anthesis (A) and silking (B), reduced 50 kernel weight (C) while no significant differences were observed for plant height (not shown). 
Cob diameter (D) showed the only significant $\textit{Inv4m}$ genotype dependency, resulting in thinner cobs under phosphorus deficiency.
Time course of stover dry weight (E) shows lower biomass accumulation under $-P$ across all sampling dates for both genotypes. 
\textit{FDR} adjusted \textit{t-test} significance: \textit{n.s.} not significant,  $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
\textbf{(F)} Gene Expression Multidimensional Scaling (MDS) plot. Samples cluster by phosphorus treatment and leaf stage.
% (F) Relative reduction in stover biomass for the two experimental predictors as percent of reference mean across the time course. 
% The overall magnitude of the -P effect (dashed line) is significantly greater than the $\textit{Inv4m}$ effect (dotted line), as confirmed by the ANCOVA main effect ($\beta = 54  \pm 13 \%$, $p = 0.0152$).
% The reference group for -P is +P, and the reference group for $\textit{Inv4m}$ is CTRL. 
% Furthermore, the -P reduction shows a significant linear dependency on time ($p = 0.027$), declining as plants mature, but the $\textit{Inv4m}$ effect does not ($p = 0.90$).
}
\label{fig:phenotypes}
\end{figure*}

\clearpage

\subsection*{Plant mineral concentrations show major responses to phosphorus starvation but only minor perturbations from the \textit{Inv4m} inversion.}

Phosphorus deficiency (-P) induced strong and conserved shifts in mineral accumulation across both genotypes, indicating that the overall ionomic response is largely shared between the \textit{Inv4m} and control lines \. Phosphorus concentrations declined sharply under -P in both stover (effect estimate $\pm$ s.e: $-1592 \pm 85$~ppm, $p = 1.13 \times 10^{-25}$) and seeds ($-672 \pm 94$~ppm, $p = 1.68 \times 10^{-8}$), accompanied by a strong increase in the seed/stover P ratio ($1.99 \pm 0.13$, $p = 1.25 \times 10^{-19}$). Zinc levels increased in stover ($6.85 \pm 1.12$~ppm, $p = 3.24 \times 10^{-7}$), while Ca rose in seed ($18.96 \pm 3.43$~ppm, $p = 4.35 \times 10^{-6}$), with corresponding changes in Zn and Ca partitioning ratios ($-0.23 \pm 0.04$, $p = 3.62 \times 10^{-6}$; and $+0.0041 \pm 0.00085$, $p = 4.17 \times 10^{-5}$, respectively). Sulfur concentrations also increased under -P in both stover ($113 \pm 21$~ppm, $p = 4.49 \times 10^{-6}$) and seed ($79 \pm 30$~ppm, $p = 2.96 \times 10^{-2}$), while Mg decreased modestly in seed ($-97 \pm 30$~ppm, $p = 7.15 \times 10^{-3}$). We found a genotype-dependent response to phosphorus for stover sulfur where \textit{Inv4m} plants accumulated less sulfur under P deficiency than the control line ( $G \times E$  interaction,  $-93.8 \pm 29.2$~ppm, $p = 6.50 \times 10^{-3}$).

Additionally, we detected additive effects of \textit{Inv4m} for  Mg and Ca.  \textit{Inv4m} plants showed reduced Mg accumulation in seeds ($-95.6 \pm 31.5$~ppm, $p = 1.09 \times 10^{-2}$) and lower Ca concentrations in stover ($-411 \pm 141$~ppm, $p = 1.38 \times 10^{-2}$), in both +P and -P conditions (\ref{fig:ionome_supp} A and B).  Together, these results indicate that \textit{Inv4m} does not broadly alter phosphorus or micronutrient homeostasis under P stress, but exerts modest effects on Ca and Mg accumulation and a specific reduction in S enrichment in stover under -P.

\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/ionome.png}
\caption{Ionomic responses of \textit{Inv4m} and control (CTRL) maize lines under phosphorus sufficiency (+P) and deficiency (-P).
Boxplots show element concentrations (A) in stover and seeds, and seed/stover ratios (E) for phosphorus (P), zinc (Zn), calcium (Ca), and sulfur (S).
\textit{t-test FDR} adjusted significance: $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
Effect sizes and exact \textit{p values} are reported in Table.}
\label{fig:ionome}
\end{figure*}

\subsection*{ Elevated triglycerides and reduced phosphoglycerolipids are driven by phosphorus starvation and leaf age, while MGDG might have an additional dependence on \invfour genotype.}

Lipid profiling shows typical changes associated with both leaf aging and phosphorus starvation ($\text{Fig}~\ref{fig:volcano}~\text{B}$).
Increased leaf stage is linked to the accumulation of \textbf{digalactosyldiacylglycerol ($\text{DGGA}$)}, notably the glycolipid \textbf{DGGA36:3} ($\log_2\text{FC}=0.67 \pm 0.15, \text{FDR}=0.044$), an accumulation consistent with enhanced lipid storage during senescence.
\textbf{Phosphorus starvation} induces a well-characterized membrane lipid remodeling response, shifting from \textbf{phosphoglycerolipids} to sugar-based glycolipids.
This process is strikingly illustrated by \textbf{PC34:2}, a key phospholipid that shows significant reduction due to the $\text{-P}$ main effect ($\log_2\text{FC}=-1.60 \pm 0.08, \text{FDR}=1.58 \times 10^{-5}$) and is further decreased by the \textbf{Leaf:-P interaction} ($\log_2\text{FC}=-0.56 \pm 0.04, \text{FDR}=0.0014$), resulting in a decrease in concentration throughout the developmental gradient that is exacerbated by the phosphorus starvation.
This widespread reduction in other phosphorus-rich membrane lipids is also seen in: \textbf{Phosphatidylethanolamines ($\text{PEs}$)}, such as \textbf{PE34:4} ($\log_2\text{FC}=-2.06 \pm 0.27, \text{FDR}=0.0067$); \textbf{Lysophosphatidylethanolamines ($\text{LPEs}$)}, with \textbf{LPE18:2} being highly reduced ($\log_2\text{FC}=-2.69 \pm 0.16, \text{FDR}=7.39 \times 10^{-5}$); and \textbf{Lysophosphatidylcholines ($\text{LPCs}$)}, seen in \textbf{LPC16:1} ($\log_2\text{FC}=-3.50 \pm 0.30, \text{FDR}=0.0007$).
Concomitantly, phosphorus starvation leads to a storage response through the accumulation of \textbf{triacylglycerols (TAGs)}, evidenced by the highly upregulated \textbf{TG50:3} ($\log_2\text{FC}=3.36 \pm 0.62, \text{FDR}=0.018$).
LION lipid enrichment analysis confirms these systemic changes, showing an extremely strong enrichment of \textbf{triacylglycerols} ($\text{FDR}=1.06 \times 10^{-11}$, $\text{ES}=0.80$) and associated \textbf{lipid storage} terms, alongside a highly significant decrease in \textbf{glycerophospholipids} ($\text{FDR}=1.46 \times 10^{-8}$, $\text{ES}=-0.60$) and \textbf{membrane components} ($\text{FDR}=4.03 \times 10^{-11}$, $\text{ES}=-0.76$).
$\text{Inv4m}$ shows no apparent high-confidence main effect on differential lipid production at this level.
The observed effects involving the $\text{Inv4m}$ genotype, including notable decreases in phosphatidylethanolamines ($\text{PE36:4}$ and $\text{PE36:6}$), decreased galactolipids ($\text{DGDG36:2}$ and $\text{MGDG36:3}$), and the enhanced accumulation of $\text{MGDG34:2}$ and $\text{MGDG34:3}$ under phosphorus starvation in older leaves, highlight a complex interaction between genotype, phosphorus availability, and leaf developmental stage.
This indicates that the genotype affects membrane lipid homeostasis, potentially modifying the balance between phospholipids and non-phosphorus containing lipids.


% Lipid profiling shows typical changes associated with both leaf aging and phosphorus starvation (Fig~\ref{fig:volcano} B) . Increased leaf stage is linked to the accumulation of digalactosyldiacylglycerol (DGGA), particularly DGGA36:3 and DGGA42:1, as well as a significant increase in triacylglycerols (TGAs).
% The accumulation of TGAs suggests enhanced lipid storage, possibly as an energy reservoir or stress adaptation mechanism during senescence.Phosphorus starvation induces a well-characterized membrane lipid remodeling response, shifting from phosphoglycerolipids (e.g., phosphatidylcholines [PCs], phosphatidylethanolamines [PEs], lysophosphatidylethanolamines [LPEs], lysophosphatidylcholines [LPCs], and phosphatidylglycerols [PGs]) to sugar-based glycolipids such as DGGA. This shift is a widely observed adaptive mechanism in plants under phosphorus limitation, reducing reliance on phosphorus-rich membrane lipids while maintaining membrane integrity and function.
% Additionally, phosphorus starvation also leads to an accumulation of triacylglycerols (TGAs), which may serve as an alternative storage form of fatty acids under stress conditions.
% \invfour shows no apparent effect on the differential gene expression or lipid production at this level.
% Based on this more sensitive analysis with local false sign rate (\textit{lfsr)} , \invfour does indeed have significant effects on lipid profiles when leaves are categorized by age (bottom/older vs top/younger). The \invfour genotype shows consistent alterations across both leaf age groups, with notable decreases in phosphatidylethanolamines (PE36:4 and PE36:6), decreased galactolipids (DGDG36:2 and MGDG36:3), and a widespread increase in nearly all detected triacylglycerols except TG56:2.
% Most intriguingly, the data reveals a complex interaction between \invfour genotype, phosphorus availability, and leaf developmental stage. Under phosphorus starvation, \invfour older leaves specifically show enhanced accumulation of monogalactosyldiacylglycerols (MGDG34:2 and MGDG34:3) and most triacylglycerols (except TG56:2). However, younger leaves of the same genotype exhibit an opposite response pattern, with decreased levels of these same lipid species. This developmental stage-dependent response suggests that \invfour alters lipid remodeling mechanisms in a tissue-specific manner, particularly influencing how plants manage membrane composition and storage lipid accumulation during phosphorus limitation.
% The consistent decrease in specific phosphatidylethanolamines (PEs) across leaf stages in \invfour plants (Fig~\ref{fig::lipid_old_young}), coupled with altered galactolipid profiles, indicates that this genotype affects membrane lipid homeostasis, potentially modifying the balance between phospholipids and non-phosphorus containing lipids.
% Meanwhile, the widespread enhancement of triacylglycerol accumulation suggests \invfour may promote carbon partitioning toward storage lipids, which could represent an adaptive response to altered resource allocation.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \begin{figure*}[!ht]
% \centering
% \includegraphics[width=0.95\textwidth]{figs/design_responses.png}
% \caption[Transcriptomic responses to phosphorus starvation]{\textbf{Transcriptomic responses to phosphorus starvation}. 
%  \textbf{(C)} Gene Expression Multidimensional Scaling (MDS) plot. Samples cluster by phosphorus treatment and leaf stage.
%  \textbf{(D-E)} Differential gene expression Manhattan plots showing the statistical significance for two experimental predictors. The number of differentially expressed genes ($\textit{FDR}<0.05$, red horizontal line) is indicated.
% \textbf{(D)} Effect of the -P treatment, the top DEG \textit{pilncr-1}, precursor of \textit{mir399}, a master regulator of phosphorus starvation response. 
%  \textbf{(E)} Effect of the interaction between the $\textit{Inv4m}$ genotype and the -P treatment ($\textit{Inv4m} \times \text{-P}$), the 3 DEGs \textit{aldh2}, \textit{gras80} and \textit{flz22} are closely linked to \textit{Inv4m}.
%  \textbf{(F)} Zoom on \textit{aldehyde dehydrogenase2}, \textit{aldh2} that which lies 1.8 Mb upstream the start of  \textit{Inv4m} (position 172883881 bp)  with linked GWAS hits from MaizeGDB.
% }
% \label{fig:design}
% \end{figure*}

\subsection*{Transcriptomic responses to phosphorus starvation}

A multidimensional scaling (MDS) of of gene expression (as $log_2[\text{CPM}]$, counts per million) captured 38\% variance in the first two dimensions (Fig~\ref{fig:design} C, Supplementary file).
The first dimension alone explained 26\% of variance and is correlated to phosphorus treatment (Pearson $r=0.50$, \textit{t-test} $\textit{FDR} = 6.15 \times 10^{-4}$).
Phosphorus (P) starvation led to a global transcriptional response with a total of of 10,606 differentially expressed genes (DEGs, $\textit{FDR} < 0.05$) out of the 24011 detected in the sampled leaves.
The core of the response involved the classic mechanisms of P mobilization and reallocation, which is conserved across plant species (Table~\ref{table::phosphorusDEGs}, Fig~\ref{fig:volcano} A).
The upregulated protein-coding genes showed enrichment in cellular response to phosphate starvation (Fisher's exact test, \textit{FDR} $=9.07 \times 10^{-11}$) (Fig~\ref{fig:volcano} D).
Top DEGs known to be upregulated under P starvation included $\textit{pap19}$ ($-\log_{10}{FDR}=9.7$, $log_2{FC}=5.99$), encoding a purple acid phosphatase that hydrolyzes organic P compounds;
$\textit{pilncr1}$ ($-\log_{10}{FDR}=9.6$, $log_2{FC}=7.34$), a P deficiency-induced long non-coding $\text{RNA}$ and precursor to $\textit{miR399}$ (a master regulator of P homeostasis); and $\textit{ips1}$ ($-\log_{10}{FDR}=9.3$, $log_2{FC}=7.08$) which  is a decoy target for $\textit{miR399}$ that prevents it from repressing the \textit{PHO2} transporter, thereby enhancing P uptake efficiency.
The P-starvation response also involved modification of leaf membrane lipids.
Other upregulated genes included in the overrepresented KEGG set were: several \textit{SPX} family transcription factors, the phosphate transporters \textit{phos1}, \textit{pht1} and \textit{pht7}, which facilitate phosphate uptake and redistribution; and the purple acid phosphatases \textit{pap1} and \textit{pap14} that increase phosphorus remobilization.
We also identified an upregulated set of enzymes involved in the process of substituting phospholipids with galactolipids, supported by the enrichment of Glycerophospholipid metabolism pathway in KEGG (Fig~\ref{fig:volcano} C) and galactolipid biosynthetic process in GO (Fig~\ref{fig:volcano} C)  respectively.
% Add  KEGGenrichemnt stats galactolipid biosynthetic process
This set included the monogalactosyldiacylglycerol synthase $\textit{mgd2}$ (\textit{Zm00001eb034810}, $-\log_{10}{FDR}=10.69$, $log_2{FC}=4.83$), the glycerophosphodiester phosphodiesterase $\textit{gpx1}$ ($-\log_{10}{FDR}=9.2$, $log_2{FC}=6.48$) and the glutathione peroxidase $\textit{glpx2}$ ($-\log_{10}{FDR}=4.5$, $log_2{FC}=7.01$). 
Conversely, genes associated with phosphorus-intensive processes and photosynthesis were downregulated.
This includes $\textit{peamt2}$ (\textit{Zm00001eb294690}, $-\log_{10}{FDR}=4.93$, $log_2{FC}=-6.81$), a phosphoethanolamine $\text{N}$-methyltransferase involved in phospholipid synthesis.
Furthermore, the photosynthetic machinery was repressed, indicated by the downregulation of $\textit{rca3}$ ($-\log_{10}{FDR}=3.8$, $log_2{FC}=-3.40$), which encodes $\textit{RUBISCO}$ activase, reflecting a $\text{P}$-deficiency-induced reduction in carbon fixation capacity.
This systemic reduction in photosynthesis is also supported by the overrepresentation of the Photosynthesis antenna proteins in $\text{KEGG}$, exemplified by the downregulation of the \textit{light harvesting chlorophyll a/b binding protein10} gene ($\textit{lhcb10}$).
Multiple transcription factors such as $\textit{zim25}$ ($-\log_{10}{FDR}=4.2$, $log_2{FC}=-3.01$), $\textit{nactf132}$ (\textit{Zm00001eb324550}, $-\log_{10}{FDR}=4.47$, $log_2{FC}=-4.66$), and $\textit{bzip81}$ ($-\log_{10}{FDR}=2.8$, $log_2{FC}=-3.37$) were also repressed, suggesting a broad transcriptional reprogramming that redirects the plant resources.


\begin{figure*}[!ht]
\centering
\includegraphics[width=0.6\linewidth]{figs/volcano.png}
\caption{Transcriptomic and lipidomic responses to phosphorus deficiency and leaf developmental stage.
(\textbf{A}) \textbf{Volcano Plots of Transcriptomic Main Effects}. The left plot illustrates the main transcriptional effect of \textbf{Phosphorus deficiency} ($-\text{P}$ treatment). A total of \textbf{794} high-confidence DEGs were identified. This response highlights key P-starvation mechanisms:
* \textbf{Upregulated genes} promoting P mobilization and signaling include $\textit{pap19}$ ($\log_2\text{FC}=5.99$), a purple acid phosphatase; and $\textit{pilncr1}$ ($\log_2\text{FC}=7.34$), a precursor to the master regulator $\textit{miR399}$. The lipid-remodeling enzyme $\textit{mgd2}$ ($\log_2\text{FC}=4.83$) was also highly upregulated.
* \textbf{Downregulated genes} include $\textit{peamt2}$ ($\log_2\text{FC}=-6.81$), involved in phospholipid synthesis, and $\textit{rca3}$ ($\log_2\text{FC}=-3.40$), encoding the $\textit{RUBISCO}$ activase, indicating a systemic repression of photosynthesis.
The right plot illustrates the main transcriptional effect of \textbf{Leaf Stage} (per-stage increase). A total of \textbf{1,431} high-confidence DEGs were identified. Key genes related to development include $\textit{umc1690}$ (Transcription factor PCF7), $\textit{ntf2}$ (NTF2 domain-containing protein), and $\textit{sgrl1}$ (Protein STAY-GREEN LIKE), all significantly downregulated. The x-axis represents the $\log_{2}(\text{Fold Change})$ and the y-axis represents the $-\log_{10}(\text{P-value})$ (or $\text{FDR}$).
(\textbf{B}) \textbf{Volcano Plots of Lipidomic Main Effects}. The left plot illustrates the main lipidomic effect of \textbf{Phosphorus deficiency} on lipid species abundance. A total of \textbf{23} high-confidence DELs were identified. The most significantly downregulated lipids (indicating the membrane remodeling response) include the phospholipids \textbf{PC34:2} ($\log_2\text{FC}=-1.60$), \textbf{LPE18:2} ($\log_2\text{FC}=-2.69$), \textbf{LPC16:1} ($\log_2\text{FC}=-3.50$), \textbf{PC32:2} ($\log_2\text{FC}=-2.58$), and \textbf{PG32:0} ($\log_2\text{FC}=-1.61$). The right plot illustrates the main lipidomic effect of \textbf{Leaf Stage} on lipid species abundance. A total of \textbf{11} high-confidence DELs were identified. Top affected lipids include $\textbf{LPC18:3}$ (upregulated, $\log_2\text{FC}=1.51$), $\textbf{DG36:4}$ (downregulated, $\log_2\text{FC}=-0.76$), and $\textbf{LPE18:3}$ (upregulated, $\log_2\text{FC}=1.32$). The axes and thresholds are analogous to those in Panel A, highlighting lipids whose abundance is significantly altered by each factor independently of the other.
}
\label{fig:volcano_multiomics}
\end{figure*}


\begin{figure*}[!ht]
\centering
\includegraphics[width=0.8\linewidth]{figs/enrichment.png}
\caption{Controlled vocabulary enrichment. Over representation analysis for (A) Gene ontology Biological Process and (B) Kegg Pathways.
}
\label{fig:erichment}
\end{figure*}

\subsection*{Phosphorus Starvation Accelerates Transcription of Senescence Genes in Maize Leaves}

To understand the interplay between leaf development and nutrient stress, we first established the baseline transcriptional signatures of leaf aging.
We observed significant, opposing correlations between leaf stage and the expression of key biological processes (Fig.~X A).
 Global expression indices significant linear developmental trends with leaf age: chlorophyll synthesis declined across leaf stages ($R^2 = 0.468$, $p < 0.001$) while chlorophyll degradation increased ($R^2 = 0.282$, $p < 0.001$), with parallel trends in photosynthesis ($R^2 = 0.572$, $p < 0.001$, decreasing) and senescence markers ($R^2 = 0.277$, $p < 0.001$, increasing). 
These developmental trajectories were exemplified by declining expression of photosynthetic genes \textit{pep1} (phosphoenolpyruvate carboxylase) and \textit{ssu1} (RuBisCO small subunit) concurrent with upregulation of senescence-associated genes including \textit{Salt homolog 1} and \textit{mir3} (Fig.~X B). 
Notably, the STAY-GREEN homologs \textit{sgrl1} and \textit{nye2} exhibited opposing expression patterns despite similar functional annotations, suggesting divergent roles in senescence regulation. Phosphorus deficiency significantly accelerated these developmental programs, generating 487 genes with significant leaf stage $\times$ phosphorus interactions (Fig.~X C).
Under $-$P conditions, the divergence between anabolic and catabolic processes intensified with leaf age: chlorophyll degradation ($p < 0.01$), chlorophyll synthesis ($p < 0.05$), photosynthesis ($p < 0.001$), and senescence indices ($p < 0.01$) all showed significant interactions between developmental stage and nutrient status (Fig.~X D).
Genes with positive interaction terms, including \textit{pyruvate kinase} and \textit{Tat pathway signal sequence family protein}, showed amplified P-starvation responses in older leaves, while those with negative interactions such as \textit{glutamine dumper 3} and chlorophyll a-b binding proteins displayed dampened responses with developmental progression (Fig.~X E, F). 
This interaction pattern indicates that phosphorus deficiency not only triggers immediate metabolic adjustments but also accelerates the natural developmental program of leaf senescence, with older leaves experiencing disproportionately severe molecular stress responses that compound the effects of nutrient limitation.


\begin{figure*}[!ht]
\centering
\includegraphics[width=0.9\linewidth]{figs/leafxPinteraction.png}
\caption{\textbf{The response to phosphorus starvation increases with leaf stage and is positively correlated with indicators of leaf senescence }.
(\textbf{A}) Gene Set Transcription Indices Across Leaf Stages. Indices are calculated as the mean $\log_{10}(\text{CPM})$ for genes within defined sets and normalized across the four leaf stages to represent the proportion of the total expression range. The left panel shows Chlorophyll Synthesis (dark green) and Chlorophyll Degradation (light green/orange) sets derived from CornCyc/KEGG. Chlorophyll Synthesis shows a significant negative correlation with age ($\mathbf{R^2 = 0.468, p < 0.001}$), while Chlorophyll Degradation shows a positive correlation ($\mathbf{R^2 = 0.282, p < 0.001}$). The right panel shows Photosynthesis (dark green) and Leaf Senescence (orange) sets from GO. Photosynthesis shows a significant negative correlation ($\mathbf{R^2 = 0.572, p < 0.001}$), and Leaf Senescence shows a significant positive correlation ($\mathbf{R^2 = 0.277, p < 0.001}$). Error bars represent the standard error of the mean (SEM).
(\textbf{B}) Expression profiles ($\log_{10}(\text{CPM})$) for representative gene pairs illustrating the opposing trends of development. \textit{pep1} (phosphoenolpyruvate carboxylase, green) and \textit{ssu1} (ribulose bisphosphate carboxylase small subunit 1, green) decline as senescence-associated genes like \textit{Salt homolog 1} (orange) and \textit{mir3} (maize insect resistance 3, orange) increase. \textit{sgrl1} and \textit{nye2} are STAY-GREEN homologs that exhibit opposing expression trends, despite similar annotated functions. Error bars represent SEM.
(\textbf{C}) Volcano Plot of Leaf $\times$ Phosphorus (P) Interaction. The plot highlights genes with a significant transcriptional interaction between leaf stage and phosphorus treatment ($+\text{P}$ vs. $-\text{P}$). Genes with a negative $\log_{2}(\text{Fold Change})$ and significant $\text{FDR}$ are colored red (negative interaction), and those with a positive $\log_{2}(\text{Fold Change})$ and significant $\text{FDR}$ are colored blue (positive interaction).
(\textbf{D}) Gene Set Transcription Indices Split by Phosphorus Treatment. The mean normalized expression for Chlorophyll Synthesis/Degradation (left) and Photosynthesis/Senescence (right) is plotted for phosphorus-deficient ($-\text{P}$, dashed lines) and phosphorus-sufficient ($+\text{P}$, solid lines) conditions. All four gene sets show a significant $\text{Leaf Stage} \times \text{P-treatment}$ interaction: Chlorophyll Degradation ($\mathbf{p < 0.01}$); Chlorophyll Synthesis ($\mathbf{p < 0.05}$); Photosynthesis ($\mathbf{p < 0.001}$); and Senescence ($\mathbf{p < 0.01}$). Error bars represent SEM.
(\textbf{E}) Individual Gene Trajectories for Leaf $\times$ Phosphorus Interaction. Genes are partitioned based on their interaction term (Negative: left, Positive: right). Thin lines represent individual gene expression profiles (centered $\log_{10}(\text{CPM})$) across the four leaf stages under $-\text{P}$ (purple) and $+\text{P}$ (yellow) conditions. Bold lines illustrate the mean trend for each group.
(\textbf{F}) Expression profiles ($\log_{10}(\text{CPM})$) for representative genes from the interaction set. The upper row highlights genes with a Positive leaf $\times$ P interaction (e.g., \textit{Pyruvate kinase} and \textit{Tat pathway signal sequence family protein}), where the effect of $-\text{P}$ is amplified in older leaves. The lower row highlights genes with a Negative interaction (e.g., \textit{Glutamine dumper 3} and \textit{Chlorophyll a-b binding protein, chloroplastic}), where the effect of $-\text{P}$ is dampened in older leaves. Error bars represent SEM.
}
\label{fig:leaf_physiology_interaction}
\end{figure*}




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}

Our multi-omics analysis demonstrates that phosphorus starvation elicits a conserved molecular and phenotypic response in maize. Reduced growth, delayed flowering, and yield penalties were paralleled by transcriptomic activation of phosphate scavenging and recycling pathways, lipid remodeling, and nutrient redistribution. Importantly, these responses were consistent across genotypes differing at the \invfour inversion. Nonetheless, we identified a small set of genotype-by-phosphorus interactions exceeding significance thresholds. In transcriptomics, these outliers overlapped with loci previously associated with flowering time and plant height. In lipidomics, phosphatidylethanolamine remodeling showed genotype specificity. Such secondary GxE effects suggest that while the phosphorus starvation program is globally robust, specific genetic variants can modulate its fine-scale execution.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Materials and methods}

\subsection*{\invfour Near Introgressed Lines, growth conditions, experimental design, and phenotype measurements}

To measure the effects of the \invfour in plant field phenotypes and their phosphorus starvation response transcriptome, we used a highland traditional variety carrying the Highland haplotype of \invfour corresponding to the inverted karyotype.
The accession Michoacán 21 (referred to as Mi21), from the Mexican Cónico group, was obtained from the International Maize and Wheat Improvement Center (CIMMYT). 
In contrast, the reference genome of the temperate inbred B73, the recurrent parent for introgression, carries the lowland haplotype corresponding to the standard non-inverted karyotype at \invfour.
From the cross of Mi21 with B73 one F1 individual was backcrossed to B73 for six generations. We selected lines carrying  \invfour with a diagnostic SNP during each cycle using a cleaved amplified polymorphic sequence (CAPS) marker. 
The marker SNP is PZE04175660223 located at position 4:181637780 in the NAM B73v5 \textit{Zea mays} genome assembly.
Amplification of the polymorphic site was done with the following primer pair: \textit{CTGAGCAGGAGATGATGGCCACTC} and \textit{GGAAAGGACATAAAAGAAAGGTGCA}, and subsequently cleaved by \textit{HinfI}.
Plants were genotyped using the CASP marker for selecting heterozygous plants at BC6S2 after selfing seeds of \invfour and CTRL homozygous individuals were selected for the field trial.

Plants were planted on May 26 2022 at the Russell E. Larson Agricultural Research Farm in Rock Springs, Pennsylvania (40°42’36" N 77°57’0" W, 366 m.a.s.l.) in soil classified as a Hagerstown silt loam (fine, mixed, semiactive, mesic Typic Hapludalf).
Experimental conditions were similar to previously described \cite{strock2018}. 
The experiment had a complete block design with two phosphorus (P) levels. 
Low-P fields (5 ppm Melich-3 Phosphorus) and high-P fields (36 ppm Melich-3 Phosphorus) were divided into smaller blocks. 
Three rows per block were planted with a mean stand count of 8 plants per plot, and the plants from the center row were selected for measurements to avoid border effects. 
Fields received fertilization based on treatment requirements. 
Drip irrigation was provided during dry periods. 
Each genotype was replicated four times within its P treatment.

\subsection*{Phenotype analysis}
For stover mass growth curves, a different plant at each time point 40, 50, 60, and harvest, 121 days after planting (DAP), was collected, dried, and weighed for the same row. 
Stover dry mass data was fitted to a logistic growth model using the R package \textit{Growthcurver} \cite{sprouffske2016}.
Maximum Stover dry weight was estimated to be the maximum over the four-time points and not dry weight at harvest.
Ear measurements were taken for one ear per row at harvest. 
We modeled the individual phenotypes with the \textit{R nlme} function as the response variable in a mixed effects model with spatial structure. For each phenotype $y$ we have: 

\begin{eqnarray}
\label{eq:pheno_model}
y_{ijkr} = \beta_{0} + \beta_{1}\text{P}_i + \beta_{2} \textit{\invfour}_j + \beta_{3}[\textit{\invfour} \times \text{P}]_{ij} + u_k + \varepsilon_{ijkr}
\end{eqnarray}

Where the phenotype observation $y_{ijkr}$ corresponds to the plant $r$ in phosphorus treatment $i$ with genotype $j$ in block $k$. The fixed effects coefficients
$\beta_{0}$ for the overall mean,
$\beta_{1}$ for the effect of phosphorus treatment  $i$,
$\beta_{2}$ for the fixed effect of genotype $j$.
One random effect of the block $k$ $(u_{k}) \sim N(0, \sigma_u^2)$, and the residuals  $(\varepsilon_{ijkr}) \sim N(\mathbf{0}, \sigma_\varepsilon^2 \mathbf{\Sigma})$.
We added a correlation structure of the residuals $\mathbf{\Sigma}$ given by a spherical model \cite{pinheiro2000}:


\begin{eqnarray}
\label{eq:sp_model}
\Sigma_{pq} = \begin{cases}
1 & \text{if } d_{pq} = 0 \\
1 - \frac{3d_{pq}}{2\rho} + \frac{d_{pq}^3}{2\rho^3} & \text{if } 0 < d_{pq} < \rho \\
0 & \text{if } d_{pq} \geq \rho
\end{cases}
\end{eqnarray}

Where $d_{pq}$ is the Euclidean distance between plots $p$ and $q$ in the field (based on row and column coordinates), 
and $\rho$ is the range parameter of the spherical model.
We corrected for multiple hypotheses ($H_0: \beta = 0 $) by reporting \textit{t-tests} for the fixed effects  below \textit{FDR} = 0.05. 



\subsection*{Tissue sampling, RNA extraction, and sequencing}
We sampled the plants at 63 DAP when we estimated them to be between v10 to v12 developmental stages. 
We took tissue from the first leaf with a fully developed collar, or the first below the flag leaf, and from there on every other leaf below for a total of four sampled leaves per plant.
These leaves were numbered sequentially from 1 (most apical) to 4 (most basal).
We used four replicate plants per combination of  P treatment and \invfour genotype for a total of 64 tissue samples. 
We took ten disc samples from the leaf tips with a tissue puncher and immediately froze the tissue in 1.5 mL tubes with two steel beads precooled with liquid nitrogen and kept in dry ice until stored at -80\textdegree C.
We extracted total RNA with the QIAGEN RNAeasy Plant Mini Kit  RNA extraction kit following manufacturer procedures (QIAGEN 74904), and RNA samples were quantified in nanodrop and sent to the NCSU Core Genomics Laboratory for sequencing.
Following QC in Bioanalyzer, Illumina libraries were prepared and sequenced in a lane of Novaseq according to manufacturer recommendations.


\subsection*{Plant genotyping}
We followed \cite{brouard2022} for GATK-based RNAseq genotyping of 15 plant samples represented by 60 leaf libraries. 
Briefly, Illumina short reads were mapped to the NAM5 Zea mays B73 genome \cite{hufford2021} using \textit{STAR} \cite{dobin2013}, then we marked duplicates in the resulting BAM alignments, split reads at intron-exon junctions and recalibrated sequence quality per leaf library.
At this point, we used HaploytypeCallerfor for generating gvcfs per plant identified by field row id ($\sim 4$ libraries per plant). 
We did joint sample genotyping afterward with \textit{genotypeGVCFs}. 
Then we filtered for variant quality ( \textit{window 35, cluster, QD < 2.0, FS > 30.0, SOR > 3.0, MQ < 40.0}) for the genotypes and $50\%$ marker completion for individuals. 
This resulted in 200000 markers with $85\%$ complete data for 13 plants.
Finally, we used TASSEL5 K Nearest Neighbour imputing, producing a matrix of 19668 markers at $99.84\%$ completion. 
Shell scripts are  available at the  \href{https://github.com/sawers-rellan-labs/inv4RNA}{inv4RNA github repository}


\subsection*{Differential gene expression and differential lipid analysis}
We aligned reads to the maize Zm-B73-REFERENCE-NAM-5.0 genome using \textit{kallisto} \cite{bray2016}.
The alternative transcript alignment was turned into counts per gene per MB.
We used \textit{voom} to calculate variance according to gene expression levels and counts were converted to $log_2(\text{CPM})$. 
Lipid analysis followed a similar workflow, where we calculated variance weights with \textit{voom} for each lipid MS-spectra peak area and transformed it to $log_2$ scale.
We made a multivariate multiple regression for gene expression and lipid MS signal separately using \textit{limma} \cite{ritchie2015}. For the log transformed expression/signal $Y_{ijrs}$, from leaf $s$, in plant $r$, under phosphorus treatment $i$, with genotype $j$, we have:

 \begin{eqnarray}
% \label{eq:expression_model}
\begin{aligned}
Y_{ijrs} = {}& \beta_0 + \beta_{1}\text{Row}_l + \beta_{2}\text{Column}_{m} + \beta_3 \text{Leaf}_{s} +\beta_4 \text{P}_{i} \\ 
& + \beta_5 \textit{\invfour}_{j} + \beta_6 [\text{P} \times \textit{\invfour}]_{ij} + \varepsilon_{ijrs}
\end{aligned}
\end{eqnarray}
with residuals:
\begin{eqnarray}
\varepsilon_{ijrs} \sim \mathcal{N} (0,\phi\sigma^2)
\end{eqnarray}
We used the leaf stage ($\text{Leaf}_{s}$) as a numerical variable with $s \in \{1,2,3,4\}$ instead of categorical. This implies that $\beta_3$ represents the rate of change of expression with increasing leaf stage number, while the rest of the coefficients were defined as categorical in the same way as in equation \eqref{eq:pheno_model} and \eqref{eq:sp_model}. 
We adjusted the p-values for the t-tests of the linear model coefficients as false discovery rates and genes whose effect had a $FDR <0.05$ were deemed to be differentially expressed.
For phosphorus treatment ($\beta_4$) and \invfour genotype ($\beta_5$) we considered genes with an effect of  $|log_2(\text{Fold Change})| >2$ as top DEGs. 
In the case of the leaf effect, a gene was considered top DEG if $|log_2(\text{Fold Change})|>0.7$, i.e. $>2.1$ between leaf stage 1 and leaf stage 4.
R scripts and expression data are available at the  \href{https://github.com/sawers-rellan-labs/\invfourRNA}{\textit{\invfourRNA} github repository}. 


\subsection*{Gene Ontology and KEGG overrepresentation analysis}

Once we had sets of differentially expressed genes for the three predictors (leaf, -P, \invfour) and two types of gene expression response (upregulated and downregulated), we proceeded to annotate them with gene ontology terms and KEGG pathways using \textit{ClusterProfiler} \cite{yu2012, zicola2024}.  
We started with the B73 NAM v5 gene ontology annotation from \cite{fattel2024} and added GO terms for each intermediate node in the gene ontology tree using the \textit{ClusterProfiler} function \textit{buildGOmap}. 
Then we conducted gene over-representation analysis with the function \textit{compareCluster}, using as universe/background the set of 24011 genes detected in at least one good quality leaf RNAseq library. 
This function calculates the hypergeometric test for overrepresented ontology terms in the specified gene set and returns raw, and FDR-adjusted p-values.
We then manually reviewed the combined 1700  significant (\textit{FDR} $<0.05$) overrepresented GO term associations for the 6 predictor/regulation combinations, and we selected for illustration an \textit{ad hoc} subset with low semantic redundancy.
Similarly, We tested for KEGG pathways over representation using the \textit{enrichKEGG} function from \textit{compareCluster}, which makes the same hypothesis tests on the NCBI REFseq annotation of the B73 NAM assembly. 
Both types of overrepresentation analysis were plotted with the package \textit{DOSE} \cite{yu2015}.


\subsection*{Filtering of \invfour DEGs by phenotype association}

As our data showed evidence of \invfour accelerating flowering time and increasing plant height, we put together a list of candidate genes associated with these two phenotypes to tease out which DEGs were likely contributors to the observed \invfour effect in these traits.
For flowering time, we started with the list of 991 genes compiled by \cite{wang2021} and  62 genes from \cite{li2023a}. 
Then we downloaded the maize data from the GWAS atlas \cite{liu2023} (\textit{gwas\_association\_result\_for\_maize.txt.gz}) and selected genes that overlapped association SNPs for the \href{https://ngdc.cncb.ac.cn/gwas/browse/ontology}{Plant Phenotype and Trait Ontology} term ``days to flowering trait" \textit{PPTO:0000155}.
For this and the following candidate gene list, we considered that a gene overlapped an association SNP if the SNP was located within the 5 kb extended range of the gene model, i.e. as described in the gff gene annotation $\pm 5$ kb.
The final source of associations for flowering time was the phenotypic plasticity study in \cite{tibbs-cortes2024}  from which we used 281 genes with significant GWAS SNPs in the columns \textit{DTS\_slope}, \textit{DTS\_intcp}, \textit{DTA\_slope}, \textit{DTA\_intcp}.
%, with the most significant SNP in this range reported in Table \ref{tab:FT_PH_candidates}.
For plant height, 27 genes from \cite{liu2023}, 1210 genes with GWAS Atlas associations for the term ``plant height" \textit{PPTO:0000126}; and  39 genes overlapping phenotypic plasticity association SNPs for \textit{PH\_slope} and \textit{PH\_intcp} \cite{tibbs-cortes2024}.
The final nonredundant list consisted of a total of 2224 candidate genes for flowering time and 1272 candidates for plant height.

% The genes responding to the introgression of \invfour into the B73 background can be divided into 3 mutually exclusive groups according to their genomic location. 
% First, there are genes that are in the inversion proper; second, those that are in the flanking regions on each side of the introgression and third, those that are located outside the introgression, as described in Table \ref{tab:DEGs_distro}.
% We are introducing a genetic perturbation in the B73 background by introgressing \invfour. 
% However, after 8 generations of selecting for the inversion there still remains 24 Mb of flanking introgression in NILs. 
% This flanking introgression spans 183 genes are are differentially expressed in the \invfour lines with respect to the control plants.

% The change in expression gene outside \invfour is a response to introgression of \invfour and flanking regions because we have experimentally swapped the genotype of the introgressed genes from the B73 reference to the Mi21 allele.

\section{Acknowledgments}
We acknowledge the support of our coffee maker that made this work possible

\bibliography{Inv4mPhosphorus}

\pagebreak

\onecolumn

\section*{Supplement}
\begin{figure}[!hb]
\centering
\includegraphics[width=\textwidth]{figs/growth_curves.png} % Assuming this is the filename
\caption{
\textbf{Maize Stover Dry Weight Growth Curves and Derived Parameters Highlight Phosphorus-Dependent Effects with No Genotype-by-Environment Interaction.}
(A) Fitted logistic growth curves for stover dry weight over time for control (CTRL) and \textit{Inv4m} genotypes under phosphorus sufficient (+P, yellow) and deficient (-P, purple) conditions. Each line represents an individual NIL plot.
(B-E) Boxplots comparing derived growth parameters for CTRL and \textit{Inv4m} genotypes under +P and -P. Phosphorus deficiency significantly reduced the Area Under the Curve (AUC) for empirical data (B) and logistic model (C), prolonged the time to reach half maximum stover weight (T$_{1/2}$) (D), and decreased the maximum stover weight (STW$_{\text{max}}$) (E).
Crucially, no significant genotype-by-phosphorus interactions were observed for any of these growth parameters, indicating that \textit{Inv4m} did not modulate the plant's response to phosphorus availability.
Furthermore, there were no significant main effects of the \textit{Inv4m} genotype on these stover grotwh parameters.
\textit{FDR} adjusted \textit{t-test} significance: \textit{n.s.} not significant, $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****).
}
\label{fig:growth} % Label for your figure
\end{figure}

\pagebreak

\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/ion_supp.png}
\caption{\textbf{Secondary Ionomic responses of \textit{Inv4m} and control maize lines under phosphorus sufficiency (+P) and deficiency (-P).}
Boxplots show element concentrations (A) in Magnesium (Mg), Manganese (Mn), Potassium (K), and Iron (Fe) in stover and seeds, and Seed/stover ratios (B) for the same four minerals.
Phosphorus deficiency ($-P$) caused a significant reduction in seed Mg (A) and seed Mn (B). No significant effects of either phosphorus treatment or the \textit{Inv4m} genotype were detected for K, Fe, or the seed/stover partition ratios (B) for any of the four elements.
\textit{t-test FDR} adjusted significance: $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
Effect sizes and exact \textit{p values} are reported in Table.}
\label{fig:ionome_supp}
\end{figure*}


\pagebreak


\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes under Phosphorus Deficiency ($\text{-P}$) with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{|c|c|p{7.5cm}|c|c|} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{|c|}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{|l|}{\textit{\textbf{Upregulated Genes}}} \\
\hline
Zm00001eb003820 & pilncr1 & pi-deficiency-induced long non-coding RNA1 & 9.0 & 7.70\\
Zm00001eb148590 & ips1 & induced by phosphate starvation1 & 9.0 & 7.10\\
Zm00001eb241920 & gpx1 & glycerophosphodiester phosphodiesterase1 & 9.0 & 6.84\\
Zm00001eb064450 & pap2 & purple acid phosphatase2 & 9.0 & 4.64\\
Zm00001eb154650 & ppa & Inorganic pyrophosphatase 1 & 9.0 & 3.06\\
Zm00001eb280120 & pfk1 & phosphofructose kinase1 & 9.0 & 2.58\\
Zm00001eb063230 & plc6 & phospholipase C6 & 9.0 & 1.90\\
Zm00001eb313760 & flz & FLZ-type domain-containing protein & 8.9 & 3.03\\
Zm00001eb370610 & rfk1 & Riboflavin kinase & 8.9 & 3.98\\
Zm00001eb007180 & gmp & Mannose-1-phosphate guanyltransferase alpha & 8.8 & 2.29\\
Zm00001eb010130 & pap19 & purple acid phosphatase19 & 8.8 & 6.09\\
Zm00001eb099420 & gmps1 & GMP synthase & 4.3 & 9.92\\
Zm00001eb019570 & spx7 & SPX domain-containing membrane protein7 & 4.1 & 8.04\\
Zm00001eb425050 & mdr1 & putative multidrug resistance protein & 3.6 & 8.23\\
Zm00001eb108800 & uam1 & UDP-arabinopyranose mutase & 3.1 & 8.72\\
Zm00001eb034810 & mgd2 & Monogalactosyldiacylglycerol synthase & 2.9 & 11.12\\
Zm00001eb388800 & ltsr1 & Low temperature and salt responsive protein & 2.3 & 9.54\\
\hline
\multicolumn{5}{|l|}{\textit{\textbf{Downregulated Genes}}} \\
\hline
Zm00001eb433900 & alla1 & allantoinase1 & 6.4 & -1.93\\
Zm00001eb211170 & toc & Translocase of chloroplast, chloroplastic & 5.9 & -1.61\\
Zm00001eb214780 & ccp19 & cysteine protease19 & 5.9 & -1.95\\
Zm00001eb070520 & bhlh148 & bHLH-transcription factor 148 & 5.8 & -2.12\\
Zm00001eb243180 & sdc & Serine decarboxylase & 5.8 & -1.74\\
Zm00001eb377880 & - & - & 5.3 & -1.63\\
Zm00001eb114780 & cfm3 & CRM family member3 & 4.9 & -1.54\\
Zm00001eb405630 & c3h & C3H transcription factor (Fragment) & 4.9 & -1.57\\
Zm00001eb377890 & snf12 & SWI/SNF complex component SNF12-like protein & 4.8 & -1.64\\
Zm00001eb248820 & - & - & 4.7 & -1.84\\
Zm00001eb294690 & peamt2 & phosphoethanolamine N-methyltransferase 2 & 3.8 & -7.17\\
Zm00001eb017120 & tps8 & terpene synthase8 & 3.3 & -4.87\\
Zm00001eb066620 & tut7 & Terminal uridylyltransferase 7 & 2.7 & -4.38\\
Zm00001eb279680 & aaap48 & amino acid/auxin permease48 & 2.3 & -4.39\\
Zm00001eb324550 & nactf132 & NAC-transcription factor 132 & 2.2 & -4.33\\
Zm00001eb292550 & sec14 & SEC14 cytosolic factor family protein / phosphoglyceride transfer family protein & 1.9 & -6.43\\
Zm00001eb410750 & - & - & 1.4 & -4.18\\
\hline
\end{tabular}
\end{table}


\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes for Leaf Stage with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{|c|c|p{7.5cm}|c|c|} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{|c|}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{|l|}{\textit{\textbf{Upregulated Genes}}} \\
\hline
Zm00001eb297390 & hir3 & hypersensitive induced reaction3 & 7.4 & 0.80\\
Zm00001eb041700 & gt & Glycosyltransferase & 7.3 & 1.09\\
Zm00001eb305330 & cyp6 & cytochrome P450 & 7.3 & 0.90\\
Zm00001eb037440 & bhlh145 & bHLH-transcription factor 145 & 7.0 & 0.89\\
Zm00001eb293310 & dnaj & DNAJ heat shock N-terminal domain-containing protein & 6.7 & 0.64\\
Zm00001eb407630 & salt1 & SalT homolog1 & 6.5 & 2.54\\
Zm00001eb275060 & - & - & 6.1 & 0.73\\
Zm00001eb098650 & trpp2 & trehalose-6-phosphate phosphatase2 & 6.0 & 1.47\\
Zm00001eb370960 & wrky111 & WRKY-transcription factor 111 & 6.0 & 0.60\\
Zm00001eb163980 & sftp & Surfactant protein B containing protein & 6.0 & 0.53\\
Zm00001eb261620 & imo & Indole-2-monooxygenase & 4.0 & 2.04\\
Zm00001eb422900 & - & - & 2.8 & 1.91\\
Zm00001eb104340 & mutl3 & MUTL protein homolog 3 & 2.3 & 1.96\\
Zm00001eb169810 & sc4mol & sphinganine C4-monooxygenase 1 & 2.2 & 1.79\\
Zm00001eb294140 & - & - & 2.1 & 1.90\\
Zm00001eb002760 & cyp78a & Cytochrome P450 family 78 subfamily A polypeptide 8 & 1.8 & 2.45\\
Zm00001eb137930 & dmas & 2'-deoxymugineic-acid 2'-dioxygenase & 1.6 & 1.89\\
Zm00001eb403420 & abc\_trans & ABC-type Co2+ transport system, permease component & 1.6 & 1.81\\
Zm00001eb054710 & chemo & Chemocyanin & 1.5 & 1.89\\\hline
\multicolumn{5}{|l|}{\textit{\textbf{Downregulated Genes}}} \\
\hline
Zm00001eb152840 & pcf7 & Transcription factor PCF7 & 7.5 & -1.48\\
Zm00001eb151160 & ntf2 & NTF2 domain-containing protein & 7.5 & -1.15\\
Zm00001eb076680 & sgrl1 & Protein STAY-GREEN LIKE, chloroplastic & 7.5 & -0.95\\
Zm00001eb038410 & ucp4 & Mitochondrial uncoupling protein 4 & 7.5 & -0.70\\
Zm00001eb329970 & tyrtr & Tyrosine-specific transport protein & 7.5 & -0.63\\
Zm00001eb182020 & mph1 & protein MAINTENANCE OF PSII UNDER HIGH LIGHT 1 & 7.5 & -0.61\\
Zm00001eb176730 & ndhb1 & photosynthetic NDH subunit of subcomplex B 1, chloroplastic & 7.5 & -0.52\\
Zm00001eb391900 & tic32 & Short-chain dehydrogenase TIC 32, chloroplastic & 7.4 & -0.60\\
Zm00001eb057540 & zmm4 & Zea mays MADS4 & 7.1 & -3.40\\
Zm00001eb154820 & chk & Choline kinase & 7.1 & -0.53\\
Zm00001eb016200 & bhlh1 & BHLH transcription factor & 6.0 & -3.62\\
Zm00001eb364940 & plt29 & Lipid-transfer protein DIR1 & 5.2 & -2.80\\
Zm00001eb214750 & zmm15 & Zea mays MADS-box 15 & 5.1 & -5.04\\
Zm00001eb320160 & alkt1 & Alkyl transferase & 4.9 & -3.82\\
Zm00001eb169010 & ccp18 & cysteine protease18 & 4.0 & -2.76\\
Zm00001eb090330 & aatr1 & amino acid transporter1 & 3.8 & -3.01\\
Zm00001eb421180 & fp3 & Farnesylated protein 3 & 3.8 & -3.23\\
Zm00001eb411680 & glu2 & beta-glucosidase2 & 2.5 & -5.12\\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes in Leaf $\times$ -P interaction, effect per increased Leaf Stage($\text{-P}$) with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{|c|c|p{7.5cm}|c|c|} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{|c|}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{|l|}{\textit{\textbf{Upregulated Genes}}} \\
\hline
Zm00001eb157810 & pk & Pyruvate kinase & 5.6 & 1.18\\
Zm00001eb376160 & mrpa3 & multidrug resistance-associated protein3 & 5.4 & 0.64\\
Zm00001eb063230 & plc6 & phospholipase C6 & 4.5 & 0.56\\
Zm00001eb144680 & rns & Ribonuclease T(2) & 4.4 & 0.61\\
Zm00001eb339870 & pld16 & phospholipase D16 & 4.3 & 0.56\\
Zm00001eb393060 & piplc & PI-PLC X domain-containing protein & 4.0 & 1.15\\
Zm00001eb148030 & gmp1 & mannose-1-phosphate guanylyltransferase1 & 3.9 & 0.69\\
Zm00001eb009430 & htm4 & Heptahelical transmembrane protein 4 & 3.9 & 0.63\\
Zm00001eb011050 & bgal & Beta-galactosidase & 3.9 & 0.53\\
Zm00001eb289800 & pah1 & phosphatidate phosphatase 1 & 3.9 & 0.58\\
Zm00001eb263160 & ring & Zinc finger (C3HC4-type RING finger) family protein & 2.9 & 2.16\\
\hline
\multicolumn{5}{|l|}{\textit{\textbf{Downregulated Genes}}} \\
\hline
Zm00001eb359280 & tat & Tat pathway signal sequence family protein & 5.6 & -0.56\\
Zm00001eb207130 & cab & Chlorophyll a-b binding protein, chloroplastic & 5.4 & -1.35\\
Zm00001eb389720 & fbpase & D-fructose-1,6-bisphosphate 1-phosphohydrolase & 5.3 & -0.81\\
Zm00001eb070520 & bhlh148 & bHLH-transcription factor 148 & 5.1 & -0.96\\
Zm00001eb212520 & psad1 & photosystem I subunit d1 & 4.6 & -0.62\\
Zm00001eb179680 & cab & Chlorophyll a-b binding protein, chloroplastic & 4.6 & -0.55\\
Zm00001eb111630 & med33a & Mediator of RNA polymerase II transcription subunit 33A & 4.4 & -0.60\\
Zm00001eb362560 & ndho1 & NADH-plastoquinone oxidoreductase1 & 4.4 & -0.58\\
Zm00001eb214780 & ccp19 & cysteine protease19 & 4.2 & -0.82\\
Zm00001eb071770 & mex1 & maltose excess protein1 & 4.0 & -0.59\\
Zm00001eb256120 &  &  & 3.8 & -1.41\\
Zm00001eb235450 & taf2n & TATA-binding protein-associated factor 2N & 3.6 & -2.07\\
Zm00001eb138960 &  &  & 2.1 & -2.11\\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\centering
\footnotesize
\caption{High-confidence differentially abundant lipids across experimental factors, annotated with IUB nomenclature and lipid class.}
\label{table::differential_lipids}
\begin{tabular}{|c|c|c|c|}
\hline
\textbf{Lipid (IUB)} & \textbf{Class} & \textbf{$-\log_{10}(\textit{FDR})$} & \textbf{$\log_2(\text{FC})$}\\
\hline
\multicolumn{4}{|l|}{\textit{\textbf{Leaf Tissue Position (Leaf)}}} \\
\hline
\multicolumn{4}{|l|}{\textit{\textbf{Upregulated Lipids}}} \\
\hline
LPC18:3 & phospholipid & 3.0 & 1.51\\
LPE18:3 & phospholipid & 2.5 & 1.32\\
PC36:6 & phospholipid & 2.5 & 0.77\\
LPE18:2 & phospholipid & 1.5 & 0.56\\
DGGA36:3 & glycolipid & 1.4 & 0.67\\
\multicolumn{4}{|l|}{\textit{\textbf{Downregulated Lipids}}} \\
\hline
DG36:4 & neutral & 2.5 & -0.76\\
DGDG34:1 & glycolipid & 1.9 & -0.56\\
DG26:0 & neutral & 1.7 & -0.67\\
PC36:1 & phospholipid & 1.6 & -0.67\\
DGGA36:4 & glycolipid & 1.5 & -0.72\\
DGDG36:1 & glycolipid & 1.3 & -5.09\\
\hline
\multicolumn{4}{|l|}{\textit{\textbf{Phosphorus Deficiency (-P)}}} \\
\hline
\multicolumn{4}{|l|}{\textit{\textbf{Upregulated Lipids}}} \\
\hline
DGGA36:4 & glycolipid & 1.8 & 1.57\\
TG50:3 & neutral & 1.8 & 3.36\\
TG54:9 & neutral & 1.7 & 2.25\\
TG50:2 & neutral & 1.7 & 2.77\\
TG52:6 & neutral & 1.7 & 2.63\\
TG56:6 & neutral & 1.7 & 12.71\\
TG52:3 & neutral & 1.4 & 2.39\\
\multicolumn{4}{|l|}{\textit{\textbf{Downregulated Lipids}}} \\
\hline
PC34:2 & phospholipid & 4.8 & -1.60\\
LPE18:2 & phospholipid & 4.1 & -2.69\\
LPC16:1 & phospholipid & 3.2 & -3.50\\
PC32:2 & phospholipid & 3.2 & -2.58\\
PG32:0 & phospholipid & 3.1 & -1.61\\
PE34:4 & phospholipid & 2.2 & -2.06\\
DG26:0 & neutral & 2.1 & -1.82\\
LPC18:3 & phospholipid & 2.1 & -2.64\\
PC32:0 & phospholipid & 1.8 & -2.35\\
PC38:6 & phospholipid & 1.7 & -2.78\\
PE34:3 & phospholipid & 1.7 & -2.20\\
LPE18:3 & phospholipid & 1.7 & -2.02\\
LPC18:2 & phospholipid & 1.6 & -3.29\\
LPE16:0 & phospholipid & 1.4 & -1.86\\
PG34:3 & phospholipid & 1.4 & -3.83\\
PE32:1 & phospholipid & 1.3 & -1.67\\
\hline
\end{tabular}
\end{table}


\end{document}


