\documentclass[9pt,twocolumn,twoside]{rilabRxiv}
% Use the documentclass option 'lineno' to view line numbers
\setlength{\marginparwidth}{2cm}
\usepackage[textsize=tiny,colorinlistoftodos]{todonotes} % comments in margins
\definecolor{cornflowerblue}{rgb}{0.39, 0.58, 0.93}
\usepackage{blindtext}


%%%%%%%Add comments in color
\newcommand{\ms}[1]{{\small \textcolor{green}{#1}}}
\newcommand{\jri}[1]{{\small \textcolor{red}{#1}}}
\newcommand{\citex}[1]{{\small \textcolor{red}{CITE(#1)}}}
\newcommand{\X}{{\textcolor{red}{X}}}
\newcommand{\mex}{\textit{mexicana}\xspace}
\newcommand{\invfour}{\textit{Inv4m}\xspace}
\newcommand{\fdrgt} {$\textrm{\textit{FDR}} > 0.05$}
\newcommand{\fdreq} {$\textrm{\textit{FDR}} = 0.05$}
\newcommand{\fdrls} {$\textrm{\textit{FDR}} < 0.05$}
\newcommand{\parv}{\textit{parviglumis}\xspace}
\newcommand{\jmjii}{\textit{jmj2}\xspace}
\newcommand{\jmjiv}{\textit{jmj4}\xspace}
\newcommand{\jmjvi}{\textit{jmj6}\xspace}
\newcommand{\jmjix}{\textit{jmj9}\xspace}
\newcommand{\arabidopis}{\textit{Arabidopsis}\xspace}

\newcolumntype{b}{X}
\newcolumntype{s}{>{\hsize=.5\hsize}X}

% Set supplement numbers to S and start counting newly
\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }


\usepackage{hyperref}
\usepackage{CJKutf8}
% \begin{CJK}{UTF8}{min}
% \verb|¯\_(ツ)_/¯|
% \end{CJK}

\title{Multi-Omics of Maize Chromosomal Inversion $\textit{Inv4m}$ in Phosphorus Deficiency Show Typical Starvation Responses and Leaf-Age Dependency, Rather Than Adaptive Contributions from the Chromosomal Inversion.}
%\title{Does maize \textit{Inv4m} contribute to adaptation to soils with low phosphorus? Not really}

\author[$1$,$2$,*]{Fausto Rodríguez-Zapata}
\author[$1$,$2$]{Nirwan Tandukar}
\author[$1$]{Ruthie Stokes}
\author[$3$]{Allison Barnes}
\author[$4$]{Sergio Pérez-Limón}
\author[$4$]{Melanie Perryman}
\author[$5$]{Miguel A. Piñeros}
\author[$6$]{Daniel Runcie}
\author[$4$]{Ruairidh Sawers}
\author[$1$,*]{Rubén Rellán-Álvarez}

\affil[$1$,*]{Department of Molecular and Structural Biochemistry, N.C. Plant Sciences Initiative, North Carolina State University, Raleigh, NC, USA.}
\affil[$2$]{Genetics and Genomics Program, North Carolina State University, Raleigh, NC, USA}
\affil[$3$]{United States Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, NC 27695}
\affil[$4$]{Department of Plant Science, Pennsylvania State University, University Park, PA, USA}
\affil[$5$]{Robert W. Holley Center for Agriculture and Health, USDA-ARS, Ithaca, NY, USA}
\affil[$6$]{Department of Plant Sciences, University of California, Davis, CA, USA}


\keywords{Local Adaptation, highland maize, Teosinte Mexicana introgression}

\runningtitle{The Role of \textit{Invm4} in adaptation to low phosphorus availability} % For use in the footer

%% For the footnote.
%% Give the last name of the first author if only one author;
\runningauthor{Rodríguez-Zapata}
%% last names of both authors if there are two authors;
% \runningauthor{FirstAuthorLastname and SecondAuthorLastname}
%% last name of the first author followed by et al, if more than two authors.
\runningauthor{Rodríguez-Zapata \textit{et al.}}


%%% Abstract %%%%%%%%%%%%%%%%%%
\begin{abstract}
Local adaptation of a species involves the selection of adaptive alleles that confer a fitness advantage in their local environment. Inversions prevent recombination between the standard and inverted heterozygous hybrids. Inversions can play a crucial role in local adaptation by locking together a set of co-adapted alleles, referred to as supergenes. 
\textit{Inv4m} is a 13 Mb inversion in maize that is particularly prevalent in highland maize and highland maize relatives from Mexico. Maize from the highlands of the Trans-Mexican volcanic belt has been shown to be well-adapted to the volcanic, acidic soils with low phosphorus availability. \textit{Inv4m} carries several genes involved in P acquisition and utilization. We therefore hypothesized that \textit{Inv4m} may be involved in adaptation to low phosphorus levels. To test this hypothesis, we introgressed a highland maize variety from the highlands of Michoacán, México, into the temperate line B73 and developed Near-Introgression Lines carrying \textit{Inv4m}. We then grew NILS carrying the inversion and controls in soils with low and high phosphorus and evaluated fitness effects of the inversion, as well as changes in gene expression using RNA-Seq.  
Here, we demonstrate that P starvation elicits highly conserved transcriptomic, lipidomic, and ionomic responses across near-isogenic lines that differ at the \invfour inversion. Thousands of genes, multiple lipid classes, and major nutrient pools respond strongly to P availability. However, we did not observe significant interactions with \textit{Inv4m}. Nonetheless, a small number of genotype-by-phosphorus interactions exceed statistical thresholds, suggesting secondary modulation of conserved responses. These exceptions may be linked to phenotypic variation in height and flowering, which is dependent on the \invfour karyotype, thereby connecting nutrient stress to developmental control. Our results highlight the robustness of P starvation responses and provide an entry point for dissecting outlier interactions of potential adaptive significance. 
\end{abstract}
%%%%%%%%%%%%%%%%%%%%%%%%%%


\setboolean{displaycopyright}{true}

\begin{document}

\maketitle
\thispagestyle{firststyle}
%\firstpagefootnote
\correspondingauthoraffiliation{
Department of Molecular and Structural Biochemistry, N.C. Plant Sciences Initiative, North Carolina State University, Raleigh, NC, USA.
E-mail: frodrig4@ncsu.edu, rrellan@ncsu.edu}
\vspace{-11pt}%

\setboolean{displaylineno}{true}
\ifthenelse{\boolean{displaylineno}}{\linenumbers}{}

\section{Introduction}
%(todo: fausto) Anadir sobre patrones de senescencia y respuesta a fosforo en la introduccion. Enfatizar que estos datos son de campo.

\lettrine[lines=2]{\color{color2}M}aize was originally domesticated in the tropical lowlands of Mexico. Before its expansion into temperate regions, maize was introduced to the Mexican highlands and the Southwestern United States, where sympatry with highland teosinte \textit{Zea mays ssp. mexicana} (shorthand \mex) likely facilitated the introgression of adaptive alleles from \mex. Teosinte \mex introgression probably facilitated adaptation to temperate zones and further expansion worldwide \cite{yang2023, guo2018-on, barnes2022}. 
However, while the average \mex introgression in modern maize is around 18\% \cite{yang2023}, not all highland-adaptive loci are present in temperate maize. Highland-associated chromosomal inversions, such as \invfour and \textit{Inv9f}, are prevalent in highland teosinte populations \cite{pyhajarvi2013} and traditional Mexican maize varieties (TVs) \cite{crow2020,gonzalez-segovia2019-jy} but are rare in modern temperate maize. 

Chromosomal inversions can contribute to local adaptation by preserving locally adapted alleles across multiple loci and reducing recombination within the inversion \cite{kirkpatrick2006b}. 
Genotyping of teosinte populations using the Maize 50K chip revealed that \invfour spans 13 Mb and is predominantly found in \mex populations \cite{pyhajarvi2013}. 
In Mexican TVs, variation in \invfour is associated with elevation and flowering time \cite{romero_navarro2017-cn}. 
Additionally, \invfour shows reduced genetic diversity, a clinal relationship with elevation, and is nearly fixed in locations above 2500 m.a.s.l. \cite{crow2020}. 
The inversion exhibits suppressed recombination, as confirmed in a biparental cross \cite{gonzalez-segovia2019-jy}.
\invfour demonstrates classic patterns of gene-by-environment interactions indicative of local adaptation. 
Plants carrying the \invfour-highland allele exhibit delayed flowering at low elevations and earlier flowering at high elevations \cite{gates2019-xu, barnes2022}. 
The highland haplotype of \invfour was introgressed from \textit{Zea mays ssp. mexicana} \cite{pyhajarvi2013,calfee2021-mr,hufford2013-gs}, a wild maize relative native to the Mexican highlands.

Despite strong evidence linking \invfour to local adaptation, the physiological processes and environmental factors underlying its adaptive role remain unclear. 
Furthermore, the specific genes within \invfour that drive local adaptation are largely unidentified. 
Previous research has shown that \invfour-highland upregulates photosynthesis genes in response to cold at the seedling stage \cite{crow2020} and is associated with earlier flowering in the Mexican highlands, which likely enhances fitness in environments with limited growth-degree accumulation throughout the year \cite{romero_navarro2017-cn}. 
However, cold is not the only limiting factor for plant growth in the Mexican highlands.
Volcanic soils (Andosols), which dominate the Mexican highlands, present an additional constraint. 
Approximately 95\% of natural Andosol profiles in Mexico are found above 2000 m.a.s.l. \cite{paz-pellat2018,inegi2013}. 
These soils are characterized by high phosphorus retention \cite{krasilnikov2013}, which leads to low phosphorus availability for plant uptake \cite{galvan-tejada2014}. 
MICH21, one of the Mexican highland maize accessions analyzed by \cite{crow2020}, originates from the Purépecha Plateau, where Andosols and phosphorus-efficient TVs are common \cite{paz-pellat2018,galvan-tejada2014,bayuelo-jimenez2011,bayuelo-jimenez2014}.
\invfour may contribute to adaptation in the highlands by carrying alleles that enhance the phosphorus starvation response (PSR). 
For example, the phosphate transporter gene \textit{ZmPho1;2a}, located within \invfour, is a strong candidate for adaptation to low phosphorus availability \cite{salazar-vidal2016-rl, Ma2021-zf}.

In this study, we aimed to understand the physiological and molecular effects of \invfour and to identify candidate genes within the inversion that could elucidate its adaptive role. 
Specifically, we tested whether \invfour-highland contributes to adaptation to low phosphorus availability. To achieve this, we backcrossed MICH21, a Mexican highland TV carrying \invfour, into the B73 genetic background for eight generations, generating Near Isogenic Lines (NILs). 
These NILs were grown under temperate field conditions with two phosphorus treatments to evaluate flowering time, height, and transcriptomic responses.

We observed that \invfour significantly reduces flowering time and increases plant height, independent of phosphorus levels. We identified a cluster of JUMONJI methyltransferases, which have higher copy numbers in modern maize compared to highland teosinte and TVs, as potential contributors to flowering time differences. Additionally, we found a group of cell cycle-related genes that may underlie height differences. 
While classical PSR genes were identified within the inversion, \invfour had no detectable effect on the phosphorus starvation response. 
These findings provide insights into the genetic mechanisms underlying \invfour’s contribution to local adaptation and highlight potential candidate genes driving its effects.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}


\subsection*{Phosphorus Starvation Dominates Maize Phenotypic Effects with \textit{Inv4m} Dependency Restricted to Cob Diameter.}

Phosphorus deficiency delayed flowering, reduced biomass, and diminished yield  (Fig.~\ref{fig:phenotypes}) in both control (CTRL) and  \textit{Inv4m} lines.
Under $-P$ conditions, anthesis and silking occurred more than three days later relative to $+P$ (Fig.~\ref{fig:phenotypes}~A and B; anthesis: $3.60 \pm 0.26$~days, $p = 2.3 \times 10^{-20}$; silking: $3.42 \pm 0.23$~days, $p = 5.7 \times 10^{-22}$; marginal effect estimate $\pm$ S.E, \textit{FDR} adjusted \textit{p-value}).
In reduced phosphorus, the 50-kernel weight decreased by nearly $18\%$ ($-1.86 \pm 0.37$~g, $p = 8.2 \times 10^{-6}$, Fig.~\ref{fig:phenotypes}~C).
Biomass accumulation was diminished under phosphorus starvation at all measured time points (Fig.~\ref{fig:phenotypes}~E): stover dry weight declined by $5.09 \pm 0.71$~g at 40~DAP ($p = 1.3 \times 10^{-9}$), $16.45 \pm 1.11$~g at 50~DAP ($p = 1.8 \times 10^{-20}$), and $27.94 \pm 3.47$~g at 60~DAP ($p = 1.4 \times 10^{-10}$).
By harvest time (121~DAP), stover biomass remained around $18.5\%$ lower ($-18.70 \pm 3.27$~g, $p = 3.9 \times 10^{-7}$).
Fitted logistic growth curves captured the effect of P-starvation on multiple model parameters (Fig.~\ref{fig:growth}). -P treatment significantly reduced the area under both the empirical (AUCE; $-1.96 \pm 0.18~\text{kg} \times \text{day}$, $p=2.2 \times 10^{-14}$) and logistic growth curves (AUCL; $-1.80 \pm 0.19~\text{kg} \times \text{day}$, $p=2.0 \times 10^{-12}$). Additionally, it reduced the fitted maximum stover weight ($\text{STW}_{\text{max}}$) by approximately $23.00 \pm 3.26$ g ($p = 5.6 \times 10^{-9}$) and delayed the time to reach half maximum stover weight ($\text{T}_{1/2}$) by $3.49 \pm 0.80$ days ($p = 6.0 \times 10^{-5}$), relative to the $+\text{P}$ treatment. We found no significant difference in the rate of stover biomass accumulation.
These phenotypic changes match the canonical maize phosphorus starvation syndrome, indicating a robust physiological response to nutrient limitation.
Crucially, no significant genotype-by-phosphorus interactions were detected for the primary agronomic traits, implying that the \textit{Inv4m} inversion did not alter the direction or magnitude of the main phosphorus response.
Nonetheless, we observe a significant $G \times E$ interaction effect for cob diameter, a secondary reproductive trait.
Specifically, while the cob diameter of control lines was unaffected by phosphorus starvation ($0.19 \pm 0.70$ cm, $p= 0.79$), the $\textit{Inv4m}$ plants grew a cob  $10.7\%$ thinner under nutrient limitation ($-2.81 \pm 0.68$ cm, $p= 1.4 \times 10^{-4}$; conditional effect estimate $\pm$ S.E, \textit{FDR} adjusted \textit{p-value} Fig.~\ref{fig:phenotypes}~D).
Aside from cob diameter, the effects of $\textit{Inv4m}$ were independent of phosphorus conditions and smaller than those of phosphorus starvation.

In both +P and -P treatments, the \textit{Inv4m} plants flowered earlier (anthesis: $-1.31 \pm 0.26$~days, $p = 4.6 \times 10^{-6}$; silking: $-0.93 \pm 0.23$~days, $p = 1.3 \times 10^{-4}$), grew taller by $6.41 \pm 1.05$~cm ($p = 7.7 \times 10^{-8}$), and accumulated less stover biomass at harvest ($-7.94 \pm 3.27$~g, $p = 1.8 \times 10^{-2}$; Fig. Supplementary figure).
%Phosphorus deficiency, on average, caused a larger lag of stover biomass than the difference due to \textit{Inv4m} throughout the measured time span (ANCOVA main effect $\beta = 54  \pm 13 \%$, $p = 0.0152$.
We found a significant linear time dependence for -P relative reduction in biomass ($p = 0.027$) but the time effect was not significant for the inversion ($p = 0.90$) Fig.~\ref{fig:biomass}~F).
The biomass lag of -P plants relative to the +P decreased as plants matured, from $48\%$ at 40~DAP to $18\%$ at harvest.
Overall, while phosphorus starvation consistently resulted in severe reproductive and vegetative penalties the $\textit{Inv4m}$ inversion had a smaller and mostly independent influence on the plant phenotype.


\begin{figure*}[!ht]
\centering
\includegraphics[width=\textwidth]{figs/phenotypes.png}
\caption{
\textbf{Maize Response to Phosphorus Starvation is Largely Additive, with $\textit{Inv4m}$ Modulating Cob Diameter.}
\textbf{(A)} Experimental design showing the four sampled leaves from $\textit{Inv4m}$ and control (CTRL) NILs.
An increasing number corresponds to older leaves.
\textbf{(B)} Aerial view of the experimental field at Rocksprings, PA.
Boxplots show phenotypic responses of control (CTRL) and \textit{Inv4m} genotypes under phosphorus sufficiency (+P) and deficiency (-P). 
Phosphorus starvation led to delayed anthesis \textbf{(C)} and silking \textbf{(D)}, reduced 50 kernel weight \textbf{(E)} while no significant differences were observed for plant height (not shown). 
Cob diameter (F) showed the only significant $\textit{Inv4m}$ genotype dependency, resulting in thinner cobs under phosphorus deficiency.
Time course of stover dry weight \textbf{(G)} shows lower biomass accumulation under $-P$ across all sampling dates for both genotypes. 
\textit{FDR} adjusted \textit{t-test} significance: \textit{n.s.} not significant,  $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
\textbf{(H)} Stover dry weight fitted to a logistic growth model, each line corresponds to a plot.
% (F) Relative reduction in stover biomass for the two experimental predictors as percent of reference mean across the time course. 
% The overall magnitude of the -P effect (dashed line) is significantly greater than the $\textit{Inv4m}$ effect (dotted line), as confirmed by the ANCOVA main effect ($\beta = 54  \pm 13 \%$, $p = 0.0152$).
% The reference group for -P is +P, and the reference group for $\textit{Inv4m}$ is CTRL. 
% Furthermore, the -P reduction shows a significant linear dependency on time ($p = 0.027$), declining as plants mature, but the $\textit{Inv4m}$ effect does not ($p = 0.90$).
}
\label{fig:phenotypes}
\end{figure*}

\clearpage

\subsection*{Plant mineral concentrations show major responses to phosphorus starvation but only minor perturbations from the \textit{Inv4m} inversion.}

Phosphorus deficiency (-P) induced strong and conserved shifts in mineral accumulation across both genotypes, indicating that the overall ionomic response is largely shared between the \textit{Inv4m} and control lines \. Phosphorus concentrations declined sharply under -P in both stover (effect estimate $\pm$ s.e: $-1592 \pm 85$~ppm, $p = 1.13 \times 10^{-25}$) and seeds ($-672 \pm 94$~ppm, $p = 1.68 \times 10^{-8}$), accompanied by a strong increase in the seed/stover P ratio ($1.99 \pm 0.13$, $p = 1.25 \times 10^{-19}$). Zinc levels increased in stover ($6.85 \pm 1.12$~ppm, $p = 3.24 \times 10^{-7}$), while Ca rose in seed ($18.96 \pm 3.43$~ppm, $p = 4.35 \times 10^{-6}$), with corresponding changes in Zn and Ca partitioning ratios ($-0.23 \pm 0.04$, $p = 3.62 \times 10^{-6}$; and $+0.0041 \pm 0.00085$, $p = 4.17 \times 10^{-5}$, respectively). Sulfur concentrations also increased under -P in both stover ($113 \pm 21$~ppm, $p = 4.49 \times 10^{-6}$) and seed ($79 \pm 30$~ppm, $p = 2.96 \times 10^{-2}$), while Mg decreased modestly in seed ($-97 \pm 30$~ppm, $p = 7.15 \times 10^{-3}$). We found a genotype-dependent response to phosphorus for stover sulfur where \textit{Inv4m} plants accumulated less sulfur under P deficiency than the control line ( $G \times E$  interaction,  $-93.8 \pm 29.2$~ppm, $p = 6.50 \times 10^{-3}$).

Additionally, we detected additive effects of \textit{Inv4m} for  Mg and Ca.  \textit{Inv4m} plants showed reduced Mg accumulation in seeds ($-95.6 \pm 31.5$~ppm, $p = 1.09 \times 10^{-2}$) and lower Ca concentrations in stover ($-411 \pm 141$~ppm, $p = 1.38 \times 10^{-2}$), in both +P and -P conditions (\ref{fig:ionome_supp} A and B).  Together, these results indicate that \textit{Inv4m} does not broadly alter phosphorus or micronutrient homeostasis under P stress, but exerts modest effects on Ca and Mg accumulation and a specific reduction in S enrichment in stover under -P.

\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/ionome.png}
\caption{Ionomic responses of \textit{Inv4m} and control (CTRL) maize lines under phosphorus sufficiency (+P) and deficiency (-P).
Boxplots show element concentrations (A) in stover and seeds, and seed/stover ratios (E) for phosphorus (P), zinc (Zn), calcium (Ca), and sulfur (S).
\textit{t-test FDR} adjusted significance: $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
Effect sizes and exact \textit{p values} are reported in Table.}
\label{fig:ionome}
\end{figure*}

\subsection*{ Elevated triglycerides and reduced phosphoglycerolipids are driven by phosphorus starvation and leaf age.}
% While MGDG might have an additional dependence on  \invfour genotype
Lipid profiling shows typical changes associated with both leaf aging and phosphorus starvation ($\text{Fig}~\ref{fig:volcano}~\text{B}$).
Increased leaf stage is linked to the accumulation of \textbf{digalactosyldiacylglycerol ($\text{DGGA}$)}, notably the glycolipid \textbf{DGGA36:3} ($\log_2\text{FC}=0.67 \pm 0.15, \text{FDR}=0.044$), an accumulation consistent with enhanced lipid storage during senescence.
\textbf{Phosphorus starvation} induces a well-characterized membrane lipid remodeling response, shifting from \textbf{phosphoglycerolipids} to sugar-based glycolipids.
This process is strikingly illustrated by \textbf{PC34:2}, a key phospholipid that shows significant reduction due to the $\text{-P}$ main effect ($\log_2\text{FC}=-1.60 \pm 0.08, \text{FDR}=1.58 \times 10^{-5}$) and is further decreased by the \textbf{Leaf:-P interaction} ($\log_2\text{FC}=-0.56 \pm 0.04, \text{FDR}=0.0014$), resulting in a decrease in concentration throughout the developmental gradient that is exacerbated by the phosphorus starvation.
This widespread reduction in other phosphorus-rich membrane lipids is also seen in: \textbf{Phosphatidylethanolamines ($\text{PEs}$)}, such as \textbf{PE34:4} ($\log_2\text{FC}=-2.06 \pm 0.27, \text{FDR}=0.0067$); \textbf{Lysophosphatidylethanolamines ($\text{LPEs}$)}, with \textbf{LPE18:2} being highly reduced ($\log_2\text{FC}=-2.69 \pm 0.16, \text{FDR}=7.39 \times 10^{-5}$); and \textbf{Lysophosphatidylcholines ($\text{LPCs}$)}, seen in \textbf{LPC16:1} ($\log_2\text{FC}=-3.50 \pm 0.30, \text{FDR}=0.0007$).
Concomitantly, phosphorus starvation leads to a storage response through the accumulation of \textbf{triacylglycerols (TAGs)}, evidenced by the highly upregulated \textbf{TG50:3} ($\log_2\text{FC}=3.36 \pm 0.62, \text{FDR}=0.018$).
LION lipid enrichment analysis confirms these systemic changes, showing an extremely strong enrichment of \textbf{triacylglycerols} ($\text{FDR}=1.06 \times 10^{-11}$, $\text{ES}=0.80$) and associated \textbf{lipid storage} terms, alongside a highly significant decrease in \textbf{glycerophospholipids} ($\text{FDR}=1.46 \times 10^{-8}$, $\text{ES}=-0.60$) and \textbf{membrane components} ($\text{FDR}=4.03 \times 10^{-11}$, $\text{ES}=-0.76$).
$\text{Inv4m}$ shows no apparent high-confidence main effect on differential lipid production at this level.
The observed effects involving the $\text{Inv4m}$ genotype, including notable decreases in phosphatidylethanolamines ($\text{PE36:4}$ and $\text{PE36:6}$), decreased galactolipids ($\text{DGDG36:2}$ and $\text{MGDG36:3}$), and the enhanced accumulation of $\text{MGDG34:2}$ and $\text{MGDG34:3}$ under phosphorus starvation in older leaves, highlight a complex interaction between genotype, phosphorus availability, and leaf developmental stage.
This indicates that the genotype affects membrane lipid homeostasis, potentially modifying the balance between phospholipids and non-phosphorus containing lipids.


% Lipid profiling shows typical changes associated with both leaf aging and phosphorus starvation (Fig~\ref{fig:volcano} B) . Increased leaf stage is linked to the accumulation of digalactosyldiacylglycerol (DGGA), particularly DGGA36:3 and DGGA42:1, as well as a significant increase in triacylglycerols (TGAs).
% The accumulation of TGAs suggests enhanced lipid storage, possibly as an energy reservoir or stress adaptation mechanism during senescence.Phosphorus starvation induces a well-characterized membrane lipid remodeling response, shifting from phosphoglycerolipids (e.g., phosphatidylcholines [PCs], phosphatidylethanolamines [PEs], lysophosphatidylethanolamines [LPEs], lysophosphatidylcholines [LPCs], and phosphatidylglycerols [PGs]) to sugar-based glycolipids such as DGGA. This shift is a widely observed adaptive mechanism in plants under phosphorus limitation, reducing reliance on phosphorus-rich membrane lipids while maintaining membrane integrity and function.
% Additionally, phosphorus starvation also leads to an accumulation of triacylglycerols (TGAs), which may serve as an alternative storage form of fatty acids under stress conditions.
% \invfour shows no apparent effect on the differential gene expression or lipid production at this level.
% Based on this more sensitive analysis with local false sign rate (\textit{lfsr)} , \invfour does indeed have significant effects on lipid profiles when leaves are categorized by age (bottom/older vs top/younger). The \invfour genotype shows consistent alterations across both leaf age groups, with notable decreases in phosphatidylethanolamines (PE36:4 and PE36:6), decreased galactolipids (DGDG36:2 and MGDG36:3), and a widespread increase in nearly all detected triacylglycerols except TG56:2.
% Most intriguingly, the data reveals a complex interaction between \invfour genotype, phosphorus availability, and leaf developmental stage. Under phosphorus starvation, \invfour older leaves specifically show enhanced accumulation of monogalactosyldiacylglycerols (MGDG34:2 and MGDG34:3) and most triacylglycerols (except TG56:2). However, younger leaves of the same genotype exhibit an opposite response pattern, with decreased levels of these same lipid species. This developmental stage-dependent response suggests that \invfour alters lipid remodeling mechanisms in a tissue-specific manner, particularly influencing how plants manage membrane composition and storage lipid accumulation during phosphorus limitation.
% The consistent decrease in specific phosphatidylethanolamines (PEs) across leaf stages in \invfour plants (Fig~\ref{fig::lipid_old_young}), coupled with altered galactolipid profiles, indicates that this genotype affects membrane lipid homeostasis, potentially modifying the balance between phospholipids and non-phosphorus containing lipids.
% Meanwhile, the widespread enhancement of triacylglycerol accumulation suggests \invfour may promote carbon partitioning toward storage lipids, which could represent an adaptive response to altered resource allocation.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \begin{figure*}[!ht]
% \centering
% \includegraphics[width=0.95\textwidth]{figs/design_responses.png}
% \caption[Transcriptomic responses to phosphorus starvation]{\textbf{Transcriptomic responses to phosphorus starvation}. 
%  \textbf{(C)} Gene Expression Multidimensional Scaling (MDS) plot. Samples cluster by phosphorus treatment and leaf stage.
%  \textbf{(D-E)} Differential gene expression Manhattan plots showing the statistical significance for two experimental predictors. The number of differentially expressed genes ($\textit{FDR}<0.05$, red horizontal line) is indicated.
% \textbf{(D)} Effect of the -P treatment, the top DEG \textit{pilncr-1}, precursor of \textit{mir399}, a master regulator of phosphorus starvation response. 
%  \textbf{(E)} Effect of the interaction between the $\textit{Inv4m}$ genotype and the -P treatment ($\textit{Inv4m} \times \text{-P}$), the 3 DEGs \textit{aldh2}, \textit{gras80} and \textit{flz22} are closely linked to \textit{Inv4m}.
%  \textbf{(F)} Zoom on \textit{aldehyde dehydrogenase2}, \textit{aldh2} that which lies 1.8 Mb upstream the start of  \textit{Inv4m} (position 172883881 bp)  with linked GWAS hits from MaizeGDB.
% }
% \label{fig:design}
% \end{figure*}

\subsection*{Transcriptomic responses to phosphorus starvation}

A multidimensional scaling (MDS) of gene expression (as $log_2[\text{CPM}]$, counts per million) captured 38\% variance in the first two dimensions (Fig~\ref{fig:volcano_multiomics} C, Supplementary file).
The first dimension alone explained 26\% of variance and is correlated to phosphorus treatment (Pearson $r=0.50$, \textit{t-test} $\textit{FDR} = 6.15 \times 10^{-4}$).
Phosphorus (P) starvation led to a global transcriptional response with a total of of 10,606 differentially expressed genes (DEGs, $\textit{FDR} < 0.05$) out of the 24011 detected in the sampled leaves.
The core of the response involved the classic mechanisms of P mobilization and reallocation, which are conserved across plant species (Table~\ref{table::phosphorusDEGs}, Fig~\ref{fig:volcano} A).
The upregulated protein-coding genes showed enrichment in cellular response to phosphate starvation (Fisher's exact test, \textit{FDR} $=9.07 \times 10^{-11}$) (Fig~\ref{fig:volcano} D).
Top DEGs known to be upregulated under P starvation included $\textit{pap19}$ ($-\log_{10}{FDR}=9.7$, $log_2{FC}=5.99$), encoding a purple acid phosphatase that hydrolyzes organic P compounds;
$\textit{pilncr1}$ ($-\log_{10}{FDR}=9.6$, $log_2{FC}=7.34$), a P deficiency-induced long non-coding $\text{RNA}$ and precursor to $\textit{miR399}$ (a master regulator of P homeostasis); and $\textit{ips1}$ ($-\log_{10}{FDR}=9.3$, $log_2{FC}=7.08$) which  is a decoy target for $\textit{miR399}$ that prevents it from repressing the \textit{PHO2} transporter, thereby enhancing P uptake efficiency.
The P-starvation response also involved modification of leaf membrane lipids.
Other upregulated genes included in the overrepresented KEGG set were: several \textit{SPX} family transcription factors, the phosphate transporters \textit{phos1}, \textit{pht1} and \textit{pht7}, which facilitate phosphate uptake and redistribution; and the purple acid phosphatases \textit{pap1} and \textit{pap14} that increase phosphorus remobilization.
We also identified an upregulated set of enzymes involved in the process of substituting phospholipids with galactolipids, supported by the enrichment of Glycerophospholipid metabolism pathway in KEGG (Fig~\ref{fig:enrichment} B) and galactolipid biosynthetic process in GO (Fig~\ref{fig:enrichment} B)  respectively.
% Add  KEGGenrichemnt stats galactolipid biosynthetic process
This set included the monogalactosyldiacylglycerol synthase $\textit{mgd2}$ (\textit{Zm00001eb034810}, $-\log_{10}{FDR}=10.69$, $log_2{FC}=4.83$), the glycerophosphodiester phosphodiesterase $\textit{gpx1}$ ($-\log_{10}{FDR}=9.2$, $log_2{FC}=6.48$) and the glutathione peroxidase $\textit{glpx2}$ ($-\log_{10}{FDR}=4.5$, $log_2{FC}=7.01$). 
Conversely, genes associated with phosphorus-intensive processes and photosynthesis were downregulated.
This includes $\textit{peamt2}$ (\textit{Zm00001eb294690}, $-\log_{10}{FDR}=4.93$, $log_2{FC}=-6.81$), a phosphoethanolamine $\text{N}$-methyltransferase involved in phospholipid biosynthesis.
Furthermore, the photosynthetic machinery was repressed, indicated by the downregulation of $\textit{rca3}$ ($-\log_{10}{FDR}=3.8$, $log_2{FC}=-3.40$), which encodes $\textit{RUBISCO}$ activase, reflecting a $\text{P}$-deficiency-induced reduction in carbon fixation capacity.
This systemic reduction in photosynthesis is also supported by the overrepresentation of the Photosynthesis antenna proteins in $\text{KEGG}$, exemplified by the downregulation of the \textit{light harvesting chlorophyll a/b binding protein10} gene ($\textit{lhcb10}$).
Multiple transcription factors such as $\textit{zim25}$ ($-\log_{10}{FDR}=4.2$, $log_2{FC}=-3.01$), $\textit{nactf132}$ (\textit{Zm00001eb324550}, $-\log_{10}{FDR}=4.47$, $log_2{FC}=-4.66$), and $\textit{bzip81}$ ($-\log_{10}{FDR}=2.8$, $log_2{FC}=-3.37$) were also repressed, suggesting a broad transcriptional reprogramming that redirects the plant resources.


\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/volcano.png}
\caption{Transcriptomic and lipidomic responses to phosphorus deficiency and leaf developmental stage.
(\textbf{A}) \textbf{Multidimensional Scaling of Transcripts}. The MDS plot of $\log_{2}(\text{CPM})$. Leaves tend to cluster by developmental stage and P treatment, showing more differentiation between P groups with increasing age.
(\textbf{B}) \textbf{Volcano Plots of Transcriptomic Main Effects}. 
\textit{Right}: Main transcriptional effect of \textbf{Leaf Stage} (per-stage increase). 
A total of \textbf{1,431} high-confidence DEGs were identified. Key genes related to development include $\textit{umc1690}$ (Transcription factor PCF7), $\textit{ntf2}$ (NTF2 domain-containing protein), and $\textit{sgrl1}$ (Protein STAY-GREEN LIKE), all significantly downregulated. The x-axis represents the $\log_{2}(\text{Fold Change})$ and the y-axis represents the $-\log_{10}(\text{P-value})$ (or $\text{FDR}$).
\textit{Left}: Main transcriptional effect of \textbf{Phosphorus deficiency} ($-\text{P}$ treatment). 
A total of \textbf{794} high-confidence DEGs were identified.
This response highlights key P-starvation mechanisms:
\textbf{Upregulated genes} promoting P mobilization and signaling include $\textit{pilncr1}$ ($\log_2\text{FC}=7.34$), a precursor to the P response master regulator $\textit{miR399}$ and $\textit{pap19}$ ($\log_2\text{FC}=5.99$), a purple acid phosphatase.
The lipid-remodeling enzyme $\textit{mgd2}$ ($\log_2\text{FC}=4.83$) was also highly upregulated.
\textbf{Downregulated genes} include $\textit{peamt2}$ ($\log_2\text{FC}=-6.81$), involved in phospholipid synthesis.
(\textbf{C}) \textbf{ Multidimensional Scaling of lipids }. The MDS plot of $\log_{2}(\text{CPM})$. Dimension 2 tends to separate leaves by P treatment, and dimension 3 separates leaf stage 1 from the older samples.
(\textbf{D}) \textbf{Volcano Plots of Lipidomic Main Effects}.
\textit{Left}: Main effect of \textbf{Leaf Stage} on lipids.
A total of \textbf{11} high-confidence DALs were identified. 
The axes and thresholds are analogous to those in Panel A, highlighting lipids whose abundance is significantly altered by each factor independently of the other.
\textit{Right:} Main effect of \textbf{Phosphorus deficiency} on lipids.
A total of \textbf{23} high-confidence DALs were identified.
(\textbf{E}) \textbf{Abundace profiles for Top 2  Differentially Abundant Lipids} The most significantly depleted lipids (indicating the membrane remodeling response) include the phospholipids \textbf{PC34:2} ($\log_2\text{FC}=-1.60$), \textbf{LPE18:2} ($\log_2\text{FC}=-2.69$), \textbf{LPC16:1} ($\log_2\text{FC}=-3.50$), \textbf{PC32:2} ($\log_2\text{FC}=-2.58$), and \textbf{PG32:0} ($\log_2\text{FC}=-1.61$). 
}
\label{fig:volcano_multiomics}
\end{figure*}


\begin{figure*}[!ht]
\centering
\includegraphics[width=0.95\linewidth]{figs/enrichment.png}
\caption{Controlled vocabulary enrichment. Over representation analysis for (A) Gene ontology Biological Process and (B) Kegg Pathways. C) Lion lipids.
}
\label{fig:enrichment}
\end{figure*}

\subsection*{Phosphorus Starvation Accelerates Transcription of Senescence Genes in Maize Leaves}

To understand the interplay between leaf development and nutrient stress, we first established the baseline transcriptional signatures of leaf aging.
We observed significant, opposing correlations between leaf stage and the expression of key biological processes (Fig~\ref{fig:leaf_physiology_interaction} A).
Global expression indices significant linear developmental trends with leaf age: chlorophyll synthesis declined across leaf stages ($R^2 = 0.468$, $p < 0.001$) while chlorophyll degradation increased ($R^2 = 0.282$, $p < 0.001$), with parallel trends in photosynthesis ($R^2 = 0.572$, $p < 0.001$, decreasing) and senescence markers ($R^2 = 0.277$, $p < 0.001$, increasing). 
These developmental trajectories were exemplified by declining expression of photosynthetic genes \textit{pep1} (phosphoenolpyruvate carboxylase) and \textit{ssu1} (RuBisCO small subunit) concurrent with upregulation of senescence-associated genes including \textit{Salt homolog 1} and \textit{mir3} (Fig~\ref{fig:leaf_physiology_interaction} B). 
Notably, the STAY-GREEN homologs \textit{sgrl1} and \textit{nye2} exhibited opposing expression patterns despite similar functional annotations, suggesting divergent roles in senescence regulation (Fig~\ref{fig:leaf_physiology_interaction} B) Phosphorus deficiency significantly accelerated these developmental programs, generating 487 genes with significant leaf stage $\times$ phosphorus interactions (Fig~\ref{fig:leaf_physiology_interaction} C, E).
Under $-$P conditions, the divergence between anabolic and catabolic processes intensified with leaf age: chlorophyll degradation ($p < 0.01$), chlorophyll synthesis ($p < 0.05$), photosynthesis ($p < 0.001$), and senescence indices ($p < 0.01$) all showed significant interactions between developmental stage and nutrient status (Fig~\ref{fig:leaf_physiology_interaction} D).
Genes with positive interaction terms, including \textit{pyruvate kinase} and \textit{Tat pathway signal sequence family protein}, showed amplified P-starvation responses in older leaves, while those with negative interactions such as \textit{glutamine dumper 3} and chlorophyll a-b binding proteins displayed dampened responses with developmental progression (Fig~\ref{fig:leaf_physiology_interaction} F). 
This interaction pattern indicates that phosphorus deficiency not only triggers immediate metabolic adjustments but also accelerates the natural developmental program of leaf senescence, with older leaves experiencing disproportionately severe molecular stress responses that compounds the effects of nutrient limitation.


\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/leafxPinteraction.png}
\caption{\textbf{The response to phosphorus starvation increases with leaf stage and is positively correlated with indicators of leaf senescence }.
(\textbf{A}) Gene Set Transcription Indices Across Leaf Stages. Indices are calculated as the mean $\log_{10}(\text{CPM})$ for genes within defined sets and normalized across the four leaf stages to represent the proportion of the total expression range. The left panel shows Chlorophyll Synthesis (dark green) and Chlorophyll Degradation (light green/orange) sets derived from CornCyc/KEGG. Chlorophyll Synthesis shows a significant negative correlation with age ($\mathbf{R^2 = 0.468, p < 0.001}$), while Chlorophyll Degradation shows a positive correlation ($\mathbf{R^2 = 0.282, p < 0.001}$). The right panel shows Photosynthesis (dark green) and Leaf Senescence (orange) sets from GO. Photosynthesis shows a significant negative correlation ($\mathbf{R^2 = 0.572, p < 0.001}$), and Leaf Senescence shows a significant positive correlation ($\mathbf{R^2 = 0.277, p < 0.001}$). Error bars represent the standard error of the mean (SEM).
(\textbf{B}) Expression profiles ($\log_{10}(\text{CPM})$) for representative gene pairs illustrating the opposing trends of development. \textit{pep1} (phosphoenolpyruvate carboxylase, green) and \textit{ssu1} (ribulose bisphosphate carboxylase small subunit 1, green) decline as senescence-associated genes like \textit{Salt homolog 1} (orange) and \textit{mir3} (maize insect resistance 3, orange) increase. \textit{sgrl1} and \textit{nye2} are STAY-GREEN homologs that exhibit opposing expression trends, despite similar annotated functions. Error bars represent SEM.
(\textbf{C}) Volcano Plot of Leaf $\times$ Phosphorus (P) Interaction. The plot highlights genes with a significant transcriptional interaction between leaf stage and phosphorus treatment ($+\text{P}$ vs. $-\text{P}$). Genes with a negative $\log_{2}(\text{Fold Change})$ and significant $\text{FDR}$ are colored red (negative interaction), and those with a positive $\log_{2}(\text{Fold Change})$ and significant $\text{FDR}$ are colored blue (positive interaction).
(\textbf{D}) Gene Set Transcription Indices Split by Phosphorus Treatment. The mean normalized expression for Chlorophyll Synthesis/Degradation (left) and Photosynthesis/Senescence (right) is plotted for phosphorus-deficient ($-\text{P}$, dashed lines) and phosphorus-sufficient ($+\text{P}$, solid lines) conditions. All four gene sets show a significant $\text{Leaf Stage} \times \text{P-treatment}$ interaction: Chlorophyll Degradation ($\mathbf{p < 0.01}$); Chlorophyll Synthesis ($\mathbf{p < 0.05}$); Photosynthesis ($\mathbf{p < 0.001}$); and Senescence ($\mathbf{p < 0.01}$). Error bars represent SEM.
(\textbf{E}) Individual Gene Trajectories for Leaf $\times$ Phosphorus Interaction. Genes are partitioned based on their interaction term (Negative: left, Positive: right). Thin lines represent individual gene expression profiles (centered $\log_{10}(\text{CPM})$) across the four leaf stages under $-\text{P}$ (purple) and $+\text{P}$ (yellow) conditions. Bold lines illustrate the mean trend for each group.
(\textbf{F}) Expression profiles ($\log_{10}(\text{CPM})$) for representative genes from the interaction set. The upper row highlights genes with a Positive leaf $\times$ P interaction (e.g., \textit{Pyruvate kinase} and \textit{Tat pathway signal sequence family protein}), where the effect of $-\text{P}$ is amplified in older leaves. The lower row highlights genes with a Negative interaction (e.g., \textit{Glutamine dumper 3} and \textit{Chlorophyll a-b binding protein, chloroplastic}), where the effect of $-\text{P}$ is dampened in older leaves. Error bars represent SEM.
}
\label{fig:leaf_physiology_interaction}
\end{figure*}




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% \section{Discussion}

% Our multi-omics analysis demonstrates that phosphorus starvation elicits a conserved molecular and phenotypic response in maize. Reduced growth, delayed flowering, and yield penalties were paralleled by transcriptomic activation of phosphate scavenging and recycling pathways, lipid remodeling, and nutrient redistribution. Importantly, these responses were consistent across genotypes differing at the \invfour inversion. Nonetheless, we identified a small set of genotype-by-phosphorus interactions exceeding significance thresholds. In transcriptomics, these outliers overlapped with loci previously associated with flowering time and plant height. In lipidomics, phosphatidylethanolamine remodeling showed genotype specificity. Such secondary GxE effects suggest that while the phosphorus starvation program is globally robust, specific genetic variants can modulate its fine-scale execution.


\section{Discussion}
%(todo: Fausto) El primer parafo de la discusion debber retomar la driving hypothesis, repasar contecto general revisa el paper de hpc1, 2 teosintes.

Our multi-omics analysis reveals that the maize phosphorus starvation response increases monotonically in the sampled internodes below the collar, with older leaves experiencing compounded stress that integrates both nutrient limitation and natural senescence programs.
While phosphorus deficiency triggered canonical molecular and phenotypic responses across genotypes, the magnitude and character of these responses depended on leaf developmental stage, with the \invfour chromosomal inversion showing minimal modulation of these developmental interactions.

\subsection*{Phosphorus Stress Accelerates Developmental Senescence in an Age-Dependent Manner}
Our experimental design, treating leaf stage as a continuous numerical variable in the statistical model, was specifically structured to detect linear changes in phosphorus response across the developmental gradient.
Under phosphorus-sufficient conditions, we observed the expected developmental gradient: photosynthetic capacity, measured through chlorophyll synthesis gene expression, declined progressively from young to old leaves, while senescence markers increased correspondingly (Fig.~\ref{fig:leaf_physiology_interaction}~A, B).
Phosphorus deficiency did not simply shift this gradient uniformly across all leaf positions but rather amplified the divergence between anabolic and catabolic processes specifically in older leaves, creating a developmental acceleration that was largely absent in younger tissues (Fig.~\ref{fig:leaf_physiology_interaction}~D).
Our sampling strategy, which targeted leaves with fully developed collars positioned below the apical meristem, allowed us to capture this monotonic developmental progression.
This positional sampling was critical because senescence is not monotonically sequential in maize.
Chlorophyll content exhibits a bell-shaped pattern across the leaf canopy, peaking around the collar leaf during vegetative stages \cite{ciganda2008} and at the ear leaf during reproductive stages \cite{ciganda2008,wei2025}, with bilateral decreases in both younger (above) and older (below) positions.
Similarly, other physiological traits including water content \cite{gao2023}, leaf area, leaf longevity, leaf area duration \cite{authorYEAR}, and leaf nitrogen content \cite{authorYEAR} show bilateral declines from a maximum at the ear leaf.
By restricting our sampling to internodes below the collar at the V10-V12 vegetative stage, we captured the monotonically declining phase of the developmental gradient.
Had we sampled leaves above the collar, we would have encountered the ascending phase of the bell curve, where younger leaves show increasing photosynthetic capacity.
Thus, our observed linear interaction between leaf stage and phosphorus availability reflects the fortunate alignment of our sampling strategy with the specific developmental window where senescence-related traits decline monotonically with increasing leaf age.

This pattern manifested across multiple molecular levels.
At the transcriptomic level, 487 genes showed significant leaf stage by phosphorus interactions (Fig.~\ref{fig:leaf_physiology_interaction}~C), representing pathways involved in chlorophyll metabolism, photosynthetic electron transport, and nutrient remobilization.
Gene Ontology enrichment analysis revealed that these interaction genes are not randomly distributed across cellular functions but rather concentrate in specific biological processes (Fig.~\ref{fig:erichment}).
Genes downregulated with the leaf $\times$ phosphorus interaction are enriched for photosynthesis, light harvesting complex, and leaf morphogenesis, while upregulated genes are enriched for phosphate starvation response, galactolipid biosynthesis, and phosphate homeostasis.
These enrichment patterns align with previous work by He et al.(2021), who used weighted gene co-expression network analysis (WGCNA) to identify coordinated modules of proteins and transcripts responding to phosphorus limitation in maize roots and shoots.
Critically, they found that certain metabolic processes such as carbohydrate metabolism, glycolysis, and lipid metabolism show strong concordant regulation at both mRNA and protein levels, whereas ribosome biogenesis and the ubiquitin-proteasome system respond primarily through protein abundance changes without corresponding mRNA shifts.
Our leaf-age-dependent phosphorus responses fit within this multi-layered regulatory framework.
The galactolipid biosynthesis genes we identified as enriched in the positive interaction set correspond to He et al.'s finding that sulfolipid and galactolipid biosynthesis genes are among the core conserved responses to phosphorus limitation, showing upregulation at both transcriptional and translational levels across diverse maize genotypes.
Similarly, our observation that photosynthesis genes show age-dependent downregulation parallels their identification of photosynthetic modules that decrease in abundance under low phosphate, though their bulk-tissue analysis could not resolve the developmental gradient we now document.

Critically, the 487 interaction genes partition into two distinct trajectories with divergent biological meanings (Fig.~\ref{fig:leaf_physiology_interaction}~E).
Genes with negative interaction terms exhibit a switch in the sign of phosphorus-deficiency responses in older leaves compared to the CTRL lines, with an increase in magnitude.
This trajectory corresponds to the shutdown of the light-dependent photosynthetic apparatus.
KEGG pathway enrichment analysis confirms that in these interaction genes, there is overrepresentation of light-harvesting complex genes, and simultaneously a lack of overrepresentation from Calvin cycle pathways.
Representative genes with negative interactions include chlorophyll a-b binding proteins and glutamine dumper 3 (Fig.~\ref{fig:leaf_physiology_interaction}~F, lower panels), consistent with the coordinated dismantling of thylakoid light-capture machinery while preserving stromal carbon fixation capacity.
This selective shutdown parallels findings in other systems where phosphorus limitation specifically targets ATP-dependent processes in the thylakoid membrane while alternative bypass pathways maintain carbon assimilation.

Conversely, genes with positive interaction terms show amplified phosphorus-starvation responses in older leaves, corresponding to induced senescence rather than natural developmental senescence.
These genes include pyruvate kinase and Tat pathway signal sequence family proteins (Fig.~\ref{fig:leaf_physiology_interaction}~F, upper panels), both of which are associated with enhanced metabolic flux toward nutrient remobilization and export.
The GO enrichment for this positive interaction set includes amino acid transport, aging, and detoxification processes, supporting their role in accelerated senescence programs.
This bifurcation into light-harvesting shutdown versus senescence induction reveals that phosphorus deficiency does not impose a single, unified response, but rather modulates distinct regulatory programs depending on the developmental context.

The two trajectories we observe provide a finer-grained view of the coordinated responses identified through transcript and lipid analyses.
He et al. (2021) demonstrated that phosphorus limitation triggers coordinated changes across multiple functional modules: a proteome module enriched for acid phosphatases, phenylalanine ammonia lyase, and bypass pathway enzymes showed strong positive correlation with improved phosphorus use efficiency, while modules enriched for oxidative pentose phosphate pathway proteins showed negative correlations with tissue phosphorus content.
Our finding that interaction genes partition into functionally distinct trajectories suggests that these system-wide coordination patterns are further modulated by developmental age, with older leaves amplifying gene expression for certain processes (such as senescence and nutrient remobilization) while increasingly repressing others (including light-dependent photosynthesis reactions).
Moreover, the limited correlation between transcriptomic and proteomic responses documented by He et al.(2021) may help explain why our transcriptomic interaction patterns are so clear: genes whose regulation occurs primarily at the protein stability or translation level would not appear in our differential expression analysis, potentially enriching our interaction gene set for those under strong transcriptional control.
The fact that ribosome biogenesis responds to phosphorus limitation primarily through changes in protein abundance without shifts in mRNA levels is consistent with our observation that older leaves, which have reduced growth demands, can more readily downregulate the translation machinery as part of the senescence acceleration program.

The lipidomic data reinforced this developmental stratification and aligned with previous reports of phosphorus-starvation lipid remodeling.
Phosphatidylcholine PC34:2, a major membrane phospholipid, showed both a strong main effect of phosphorus deficiency ($\log_2\text{FC}=-1.60 \pm 0.08$, $\text{FDR}=1.58 \times 10^{-5}$) and a significant interaction with leaf stage ($\log_2\text{FC}=-0.56 \pm 0.04$, $\text{FDR}=0.0014$, Fig.~\ref{fig:volcano_multiomics}).
While all leaves exhibited reduced PC34:2 under low phosphorus conditions, the magnitude of this reduction increased systematically with leaf age, suggesting that older leaves prioritize phospholipid catabolism more aggressively than younger leaves.

However, our age-resolved approach reveals nuances that are obscured in bulk-tissue measurements.
While Wang et al. (2020) reported overall increases in MGDG and DGDG molar percentages in leaf tissues, we find that these shifts are not uniform across leaf developmental stages but rather concentrated in older leaves, where phospholipid depletion and galactolipid accumulation are most pronounced.
At the lipid species level, we observe dramatic reductions in phosphatidylcholines that align with Wang et al.'s findings: PC34:2 ($\log_2\text{FC}=-1.60$, $\text{FDR}=1.58 \times 10^{-5}$) and PC32:2 ($\log_2\text{FC}=-2.58$) show strong depletion under phosphorus deficiency, consistent with their report of decreased PC molar percentages in leaves.

The reduction in PC34:2 is particularly noteworthy given its established binding to ZCN8, the maize florigen ortholog.
We previously demonstrated that PC34:2 copurifies with recombinant ZCN8 protein, confirming direct lipid-protein interaction, and identified probable binding sites through molecular docking simulations (Barnes et al., 2022).
Additionally, we demonstrated that phospholipase HPC1 expression, which influences PC34:2 levels, is correlated with flowering-time variation across the maize diversity panel, with association magnitudes comparable to those of canonical flowering genes, such as ZCN8 and ZmRAP2.7.
While a causal role for PC34:2 in flowering-time control remains to be established, the depletion of PC34:2 under phosphorus deficiency in our study, combined with the age-dependent interaction term ($\log_2\text{FC}=-0.56$ for Leaf x -P interaction), raises the possibility that phosphorus stress may influence reproductive timing not only through resource limitation but also through altered availability of lipid species that interact with flowering regulatory proteins.

Across both genotypes, phosphorus deficiency delayed anthesis by 3.6 days ($p = 2.3 \times 10^{-20}$) and silking by 3.4 days ($p = 5.7 \times 10^{-22}$).
While this delay likely reflects primarily the time required to accumulate sufficient resources through enhanced nutrient remobilization from senescing leaves, the specific depletion of PC34:2—a lipid species demonstrated to bind ZCN8—suggests a plausible additional mechanism.
If ZCN8 florigen activity or stability is modulated by PC34:2 binding, as has been shown in \textit{Arabidopsis} FT interactions with PG in temperature- and location-dependent contexts, then the systematic depletion of this specific phospholipid species could potentially impair florigen signaling.

This correlation between PC34:2 depletion and flowering delay is reinforced by the coordinated downregulation of flowering-time genes with leaf age in our dataset.
The MADS-box transcription factors \textit{zmm4} (Zm00001eb057540, $\log_2\text{FC}=-3.40$, $-\log_{10}\text{FDR}=7.10$) and \textit{zmm15} (Zm00001eb214750, $\log_2\text{FC}=-5.04$, $-\log_{10}\text{FDR}=5.13$) are among the most strongly downregulated genes with increasing leaf age, consistent with the natural progression from vegetative to reproductive phase transitions that matches our sampling time.

Similarly, we detect significant reductions in phosphatidylethanolamines (PE34:4, $\log_2\text{FC}=-2.06$; PE34:3, $\log_2\text{FC}=-2.20$) and lysophospholipids (LPE18:2, $\log_2\text{FC}=-2.69$; LPC16:1, $\log_2\text{FC}=-3.50$; LPC18:3, $\log_2\text{FC}=-2.64$), matching Wang et al.'s observed decreases in PE, LPE, and LPC classes.

The behavior of lysophosphatidylethanolamines (LPEs) in our dataset diverges from the previously reported pattern in dicotyledonous species, and this divergence extends to the transcriptional regulation of phosphatidylethanolamine biosynthesis.
In horticultural crops, exogenous LPE application delays leaf senescence through inhibition of phospholipase D (PLD), thereby preserving membrane integrity and reducing ethylene production (Amaro and Almeida, 2013).
LPE-treated tomato and pepper leaves maintain higher chlorophyll content, reduced electrolyte leakage, and delayed senescence markers compared to untreated controls (Farag and Palta, 1991b; Kang et al., 2003).
This senescence-suppressing role positions LPE as a protective agent in dicots, where its accumulation would be expected to correlate with membrane preservation and delayed aging.

Our maize data present the opposite pattern at both the metabolite and transcript levels.
Under phosphorus deficiency, we observe dramatic reductions in multiple LPE species: LPE18:2 ($\log_2\text{FC}=-2.69 \pm 0.46$, $\text{FDR}=7.39 \times 10^{-5}$), LPE18:3 ($\log_2\text{FC}=-2.02 \pm 0.66$, $\text{FDR}=0.022$), and LPE16:0 ($\log_2\text{FC}=-1.86 \pm 0.70$, $\text{FDR}=0.042$).
More strikingly, LPE18:3 shows a positive association with leaf developmental stage ($\log_2\text{FC}=1.32 \pm 0.30$, $\text{FDR}=0.0036$), indicating that LPE accumulation increases as leaves progress toward senescence under our experimental conditions.
This developmental trajectory is precisely the inverse of what would be predicted if LPE functioned as a senescence suppressor in maize as it does in dicots.

The lipidomic pattern is paralleled by coordinated transcriptional suppression of phosphatidylcholine biosynthesis.
The dramatic downregulation of phosphoethanolamine N-methyltransferase 2 (peamt2/XIPOTL1, Zm00001eb294690, $\log_2\text{FC}=-6.81 \pm 0.53$, $\text{FDR}=4.93 \times 10^{-5}$) represents one of the strongest transcriptional responses to phosphorus deficiency in our entire dataset.
This enzyme catalyzes the sequential methylation of phosphoethanolamine to phosphocholine in the Kennedy pathway for PC biosynthesis (Cruz-Ramírez et al., 2004), and its severe suppression indicates a systemic shutdown of PC synthesis from PE-derived precursors.
The ortholog in Arabidopsis, XIPOTL1, has been implicated in developmental timing, with mutations causing pleiotropic effects on flowering time, root architecture, and stress responses (Cruz-Ramírez et al., 2004).
In maize, we have found that natural variation at the peamt2 locus is associated with flowering time in Mexican highland populations (Pérez-Limón et al., 2022; Barnes et al., 2021), suggesting that modulation of phospholipid biosynthesis may contribute to adaptive variation in developmental timing under environmental stress.

The coordinate suppression of peamt2 and depletion of both PC and LPE species indicates that maize prioritizes phosphorus conservation over maintenance of phospholipid biosynthetic capacity during nutrient limitation. This strategy contrasts sharply with approaches that might attempt to maintain membrane integrity through sustained phospholipid synthesis.
Instead, the data support active phospholipid catabolism as the dominant metabolic strategy. This interpretation is reinforced by the upregulation of phospholipase genes in our dataset: phospholipase C6 (\textit{plc6}, \textit{Zm00001eb063230}) shows both a main phosphorus effect and a positive leaf × phosphorus interaction ($\log_2\text{FC}=0.56 \pm 0.04$, $\text{FDR}=2.82 \times 10^{-5}$), indicating enhanced phospholipid hydrolysis specifically in older leaves under phosphorus stress. 
Similarly, \textit{phospholipase D16} (\textit{pld16}, Zm00001eb339870) exhibits a positive leaf × phosphorus interaction ($\log_2\text{FC}=0.56 \pm 0.04$, $\text{FDR}=4.85 \times 10^{-5}$), confirming that phospholipid degradation is developmentally amplified under nutrient limitation.

Lysophosphatidylethanolamine acyltransferase 2 (lpeat2, \textit{Zm00001eb227780}), which catalyzes the conversion of lysophosphatidylethanolamine to phosphatidylethanolamine in the Kennedy pathway (Gou et al., 2017), shows no significant differential expression under our experimental conditions, suggesting that PE synthesis from LPE is not actively modulated during phosphorus stress.
Similarly, choline/ethanolamine kinase 4 (\textit{cek4}, \textit{Zm00001eb313980}), which phosphorylates ethanolamine to phosphoethanolamine in the first committed step of PE biosynthesis via the CDP-ethanolamine pathway, is absent from our high-confidence differential expression set, indicating constitutive rather than stress-responsive regulation.

The coordination between PE depletion, peamt2 suppression, and phospholipase upregulation reveals a metabolic strategy distinct from the dicot pattern.
The severe downregulation of XIPOTL1/peamt2 might effectively block the conversion of PE-derived phosphoethanolamine into PC, forcing the plant to rely on existing PC pools while simultaneously catabolizing them to recover phosphorus.
Wang et al. (2020) demonstrated that maize responds to phosphorus limitation by systematically replacing phosphorus-containing membrane lipids with galactolipids, a process that requires the efficient degradation of existing phospholipids to liberate diacylglycerol for galactolipid biosynthesis.
Within this framework, LPE represents not a regulatory signal but rather a transient intermediate in the phosphorus salvage pathway. 
The fact that LPE levels decline rather than accumulate during phosphorus stress suggests highly efficient conversion of LPE to downstream products, either through complete degradation to release phosphate or through rapid acylation back to PE for subsequent hydrolysis by other phospholipases.

The functional link between peamt2/XIPOTL1 expression and flowering time (Pérez-Limón et al., 2022; Barnes et al., 2022) provides an additional layer of interpretation for the delayed flowering we observe under phosphorus deficiency. The balance between PC and its precursors has been demonstrated to influence flowering time in Arabidopsis (Nakamura et al., 2014), with perturbations in phospholipid metabolism affecting the timing of reproductive transitions. The dramatic suppression of peamt2 under phosphorus limitation, combined with the depletion of PC34:2—a phospholipid species that binds to the maize florigen ZCN8 (Barnes et al., 2022)—suggests that phosphorus stress may delay flowering through both direct resource limitation and indirect modulation of lipid-mediated signaling pathways. 

The positive association between LPE18:3 and leaf stage further challenges a simple regulatory interpretation.
If LPE were a senescence suppressor in maize, we would expect its levels to decline as leaves age and senesce, paralleling the pattern observed when exogenous LPE delays senescence in dicots.
Instead, the age-dependent increase in LPE18:3 suggests it may serve as a senescence marker in maize rather than a senescence modulator.
This interpretation aligns with the broader lipidomic context: older leaves under phosphorus stress show enhanced phospholipid degradation (as evidenced by the PC34:2 interaction term) and increased TAG accumulation, consistent with accelerated nutrient remobilization.
LPE18:3 accumulation in older leaves may simply reflect the residual products of this enhanced catabolic activity, with 18:3-containing species being preferentially retained due to their structural properties or metabolic recalcitrance.

The mechanistic basis for this divergence between maize and dicots may reflect fundamental differences in membrane lipid metabolism between monocots and dicots.
Maize, as an 18:3 plant, relies predominantly on the eukaryotic (ER-localized) pathway for galactolipid synthesis (Wang et al., 2020), whereas many dicots also utilize the prokaryotic (plastid-localized) pathway.
This difference in lipid biosynthetic architecture may alter the metabolic fate of lysophospholipid intermediates.
In the eukaryotic pathway, diacylglycerol derived from phospholipid degradation is transported to the chloroplast outer envelope for galactolipid synthesis, creating a direct metabolic flux from PE → LPE → DAG → MGDG.
The efficiency of this pathway, particularly when PC biosynthesis is blocked by \textit{peamt2} suppression, may preclude LPE accumulation, as the lipid is rapidly consumed in downstream reactions.

Critically, all studies demonstrating LPE senescence-suppressing effects have employed exogenous application of LPE to plant tissues, often at supra-physiological concentrations (50–200 mg/L).
These experiments reveal that external LPE can influence senescence pathways when supplied at high levels, but they do not necessarily demonstrate that endogenous LPE fluctuations serve this regulatory function under natural conditions.
Our data, which measure endogenous LPE levels under phosphorus stress, may better reflect the actual metabolic role of these lipids in unstressed tissues.
The depletion of LPEs we observe under phosphorus limitation is consistent with their metabolic consumption as part of the phosphorus salvage pathway, rather than their accumulation as regulatory signals.

This contrast highlights the importance of considering phylogenetic context when interpreting lipid signaling mechanisms.
The senescence-regulatory role of LPE, established primarily in horticultural dicots through exogenous application experiments, does not appear to operate in monocots like maize, where membrane remodeling during phosphorus stress follows alternative biochemical strategies prioritizing efficient phosphorus recycling over membrane preservation. The severe suppression of XIPOTL1/peamt2, representing one of the strongest transcriptional responses to phosphorus deficiency in our dataset, exemplifies this prioritization: rather than maintaining phospholipid biosynthetic capacity, maize systematically dismantles the machinery for PC synthesis, ensuring that available phosphorus is redirected toward essential nucleic acid and ATP synthesis while membrane function is maintained through phosphorus-free galactolipids.

The reduction in phosphatidylglycerol species (PG32:0, $\log_2\text{FC}=-1.61$; PG34:3, $\log_2\text{FC}=-3.83$) further validates the membrane remodeling response documented by Wang et al.
Conversely, our detection of substantial triacylglycerol accumulation (TG50:3, $\log_2\text{FC}=3.36$; TG52:6, $\log_2\text{FC}=2.63$; TG54:9, $\log_2\text{FC}=2.25$; TG56:6, $\log_2\text{FC}=12.71$) extends Wang et al.'s observations by revealing which fatty acid combinations are preferentially stored.
The particularly striking accumulation of TG56:6, with a fold-change exceeding 6000-fold, suggests that certain very-long-chain polyunsaturated TAG species may serve specialized roles in temporary lipid storage during membrane remodeling.
The fatty acid composition of these accumulated TAGs—particularly the enrichment of polyunsaturated species containing 18:3 fatty acids—directly matches the molecular species being depleted from phospholipids, most notably PC36:6 and LPC18:3.
This compositional correspondence suggests efficient metabolic channeling from membrane phospholipids to storage lipids during phosphorus stress.

The mechanism underlying this metabolic flux may involve phospholipid:diacylglycerol acyltransferase (PDAT), which catalyzes the direct transfer of acyl groups from phospholipids to diacylglycerol to form triacylglycerols, providing an alternative pathway to diacylglycerol acyltransferase (DGAT) for TAG synthesis.
Among the four annotated maize PDAT1 genes (\textit{Zm00001eb314300}, \textit{Zm00001eb342120}, \textit{Zm00001eb118700}, and \textit{Zm00001eb100310}), only \textit{Zm00001eb100310} showed a statistically significant response to leaf age ($\beta=0.103$ per leaf stage, $\text{FDR}=9.40 \times 10^{-6}$), with this upregulation slightly enhanced under phosphorus deficiency ($\beta=0.102$ for the leaf $\times$ phosphorus interaction, $\text{FDR}=0.019$).
The remaining PDAT genes showed no significant differential expression.
However, the magnitude of the \textit{Zm00001eb100310} response falls well below our high-confidence threshold ($|\beta| > 0.5$), representing only a 1.22-fold change across the full leaf developmental gradient (2$^{0.103 \times 3}$ = 1.22).
This weak transcriptional response suggests that PDAT-mediated TAG synthesis operates primarily through constitutive enzyme activity, substrate availability, or post-translational regulation rather than through transcriptional induction.
This interpretation aligns with He et al. (2021), who demonstrated that many lipid metabolism enzymes respond to phosphorus limitation primarily through changes in protein abundance, substrate availability, or post-translational regulation without corresponding large-scale mRNA shifts.

The PDAT pathway is particularly relevant for explaining the accumulation of highly unsaturated TAG species because it directly transfers intact acyl chains from phospholipids to diacylglycerol, preserving the fatty acid composition of the donor phospholipid.
This contrasts with alternative pathways involving complete phospholipid degradation followed by de novo TAG synthesis, which would require re-desaturation of fatty acids and would be energetically expensive under phosphorus limitation.
The coordinate depletion of PC36:6 ($\log_2\text{FC}$ for leaf stage effect = 0.77) and LPC18:3 ($\log_2\text{FC}$ for leaf stage effect = 1.51), coupled with the accumulation of TAGs enriched in 18:3-containing fatty acids (TG54:9, TG52:6), is consistent with PDAT-mediated direct transfer of polyunsaturated acyl chains from senescing phospholipid membranes into storage lipids.

Wang et al. (2020) noted that TAG accumulation under phosphorus starvation likely represents transient storage for fatty acids released from membrane lipid catabolism, awaiting transport to younger tissues or conversion to energy via $\beta$-oxidation.
The age-dependent amplification of this response in our dataset—where older leaves show enhanced TAG accumulation alongside more severe phospholipid depletion—supports this interpretation and suggests that PDAT-mediated lipid salvage may be particularly important in senescing tissues.
By preserving the polyunsaturated fatty acids from membrane phospholipids in TAG form, the plant may maintain a pool of essential fatty acids that can be remobilized when needed, either for membrane repair in younger leaves or for energy metabolism during periods of carbon limitation.
The fact that this metabolic strategy operates without significant transcriptional upregulation of PDAT genes indicates that the phospholipid-to-TAG flux is likely governed by substrate availability (increasing phospholipid degradation) and possibly by post-translational enzyme regulation rather than by increased PDAT gene expression.

The enhanced accumulation of digalactosyldiacylglycerol (DGDG) under phosphorus deficiency confirms that the shift from phospholipids to galactolipids is a dominant feature of the maize phosphorus starvation response, directly paralleling Wang et al.'s reported increases in DGDG.
Additionally, we detected significant accumulation of glucuronosyldiacylglycerol (DGGA36:4, $\log_2\text{FC}=1.57$, $\text{FDR}=0.018$), a phosphorus-free membrane lipid that has been documented in Arabidopsis, rice, tomato, and soybean as a component of the phosphorus starvation response.
DGGA biosynthesis requires SQDG synthase activity, and Okazaki et al. (2015) demonstrated that DGGA accumulation represents an additional membrane lipid remodeling strategy beyond the canonical phospholipid-to-galactolipid shift, further reducing cellular phosphorus demand during nutrient limitation.
The coordinate upregulation of galactolipids (DGDG) and glucuronide lipids (DGGA) in our dataset indicates that maize employs multiple parallel pathways to replace phosphorus-containing membrane lipids under deficiency conditions.

At the gene expression level, our findings validate and extend the regulatory framework established by Wang et al. (2020).
They identified three Type B galactolipid biosynthesis genes (\textit{ZmMGD2}, \textit{ZmMGD3}, and \textit{ZmDGD2}) as strongly upregulated under phosphorus starvation in both leaves and roots.
We confirm robust upregulation of \textit{mgd2} (\textit{Zm00001eb034810}, $\log_2\text{FC}=4.83$, $-\log_{10}\text{FDR}=10.69$), which Wang et al.(2020) identified as the primary Type B MGDG synthase responsible for producing MGDG via the eukaryotic pathway under phosphorus limitation.
This gene showed the highest expression induction in their study and represents the critical enzyme converting diacylglycerol to MGDG in the outer chloroplast envelope, initiating the galactolipid biosynthetic cascade.
While we detect \textit{mgd1} (Zm00001eb313310, Type A) and \textit{mgd3} (Zm00001eb258130, Type B) in our transcriptome, neither shows differential expression at our significance threshold, suggesting that \textit{mgd2} is indeed the dominant responding isoform in leaves during phosphorus starvation, consistent with Wang et al.'s quantitative PCR results.

For DGDG synthesis, Wang et al. reported upregulation of both Type A (\textit{ZmDGD1}) and Type B (\textit{ZmDGD2}) genes, with \textit{ZmDGD2} showing the stronger response.
Although \textit{dgd2} (Zm00001eb243060) is present in our dataset, it does not reach our differential expression threshold, possibly due to differences in developmental timing (V10-V12 in our study versus their earlier sampling) or the sensitivity of RNA-seq versus qPCR.
Wang et al. demonstrated that DGD2 preferentially uses MGDG generated by Type B MGD2/MGD3 to produce DGDG in the outer chloroplast envelope, suggesting a tightly coordinated enzymatic cascade where MGD2 upregulation may be sufficient to drive the pathway.

The connection between sulfoquinovosyldiacylglycerol (SQDG) synthase and glucuronosyldiacylglycerol (DGGA) biosynthesis, as established by Okazaki et al. (2015), is directly validated by our coordinated transcriptomic and lipidomic results.
We observe strong upregulation of both \textit{sqd1} (\textit{Zm00001eb347070}, $\log_2\text{FC}=1.76$, $\text{FDR}=9.0 \times 10^{-9}$) and \textit{sqd2} (\textit{Zm00001eb297970}, $\log_2\text{FC}=1.83$, $\text{FDR}=8.9 \times 10^{-9}$) under phosphorus deficiency, with an additional \textit{sqd2} paralog (\textit{Zm00001eb335670}) showing particularly robust induction ($\log_2\text{FC}=4.17$, $\text{FDR}=1.1 \times 10^{-8}$).
These transcriptional responses are accompanied by age-dependent amplification: \textit{sqd2} shows high-confidence enhanced upregulation in older leaves under phosphorus stress ($\beta=0.53$, $\text{FDR}=0.00014$), while \textit{sqd1} exhibits a significant but more modest age-dependent response ($\beta=0.47$, $\text{FDR}=0.00035$).
The coordinate induction of these enzymes directly supports DGGA biosynthesis: SQD1 catalyzes UDP-sulfoquinovose synthesis in the chloroplast stroma, while SQD2 transfers sulfoquinovose to diacylglycerol at the inner chloroplast envelope to generate SQDG, but importantly, SQD2 also possesses the catalytic activity required for DGGA synthesis.

Okazaki et al. (2015) demonstrated that \textit{Arabidopsis} plants with mutations in \textit{SQD2}, but not \textit{SQD1}, fail to accumulate DGGA under phosphorus starvation, establishing SQD2 as the essential enzyme for DGGA biosynthesis despite its designation as an SQDG synthase.
This apparent paradox is resolved by recognizing that SQD2 possesses broad substrate specificity, capable of transferring various sugar moieties to diacylglycerol depending on substrate availability and cellular context.
Under phosphorus-replete conditions, when UDP-sulfoquinovose is abundant, SQD2 primarily synthesizes SQDG; however, under phosphorus limitation, alternative UDP-sugar substrates become favorable, leading to DGGA production.
The coordinate upregulation of both SQD1 and SQD2 in our dataset, coupled with the accumulation of DGGA36:4 ($\log_2\text{FC}=1.57$, $\text{FDR}=0.018$), confirms that this phosphorus-sparing lipid strategy documented in Arabidopsis, rice, tomato, and soybean extends to maize and represents a conserved adaptation across diverse plant lineages.

The age-dependent enhancement of \textit{sqd2} expression under phosphorus stress (with high-confidence $\beta=0.53$ exceeding our threshold) parallels the developmental pattern we observed for other membrane remodeling genes, suggesting coordinated regulation of multiple phosphorus-conserving pathways in senescing leaves.
While Wang et al. (2020) reported \textit{SQD2} upregulation under phosphorus limitation in maize, they analyzed bulk tissue samples and did not resolve the age-dependent amplification we document here.
The systematic increase in SQD enzyme expression with leaf age creates a metabolic environment increasingly favorable for DGGA synthesis in older leaves, where phosphorus salvage is most critical and where phospholipid catabolism provides abundant diacylglycerol substrates for glycolipid biosynthesis.
The fact that DGGA36:4, rather than other DGGA species, shows the strongest accumulation may reflect substrate channeling from specific phospholipid degradation pathways or preferential incorporation of 36-carbon diacylglycerols into the DGGA biosynthetic route.

This dual upregulation of galactolipid biosynthesis (MGDG/DGDG via MGD2) and glucuronide lipid biosynthesis (DGGA via SQD1/SQD2) reveals that maize employs parallel membrane remodeling strategies during phosphorus limitation, maximizing the replacement of phosphorus-containing lipids through multiple independent pathways that converge on the common goal of reducing cellular phosphorus demand while maintaining membrane integrity.

Notably, we detect upregulation of phospholipid degradation enzymes that provide the diacylglycerol and phosphatidic acid substrates essential for galactolipid synthesis.
The strong upregulation of \textit{gpx1} (Zm00001eb241920, glycerophosphodiester phosphodiesterase, $\log_2\text{FC}=6.48$, $-\log_{10}\text{FDR}=8.96$) indicates enhanced breakdown of glycerophosphodiesters to liberate glycerol-3-phosphate and facilitate phospholipid turnover.
Multiple phospholipase genes show responses: \textit{plc6} (Zm00001eb063230, phospholipase C6) exhibits both main phosphorus effect and leaf $\times$ phosphorus interaction, suggesting developmental modulation of phosphatidylinositol-specific cleavage.

Conversely, phospholipid biosynthesis is actively suppressed.
The dramatic downregulation of \textit{peamt2} (Zm00001eb294690, phosphoethanolamine N-methyltransferase 2, $\log_2\text{FC}=-6.81$, $-\log_{10}\text{FDR}=4.93$) indicates reduced commitment to phosphatidylcholine synthesis via the Kennedy pathway, directly explaining the depletion of PC species we observe.
This transcriptional suppression of phospholipid biosynthesis, combined with enhanced degradation, creates the net flux toward galactolipid accumulation that Wang et al.(2020) documented at the lipid level.

Phosphatidic acid phosphatase genes that commit lipids to triacylglycerol synthesis show developmental interactions: \textit{pah1} (\textit{Zm00001eb289800}) exhibits positive leaf $\times$ phosphorus interaction ($\log_2\text{FC}=0.58$), suggesting that the decision to shunt lipids toward TAG storage versus membrane reconstruction is age-dependent and increasingly favors storage in senescing leaves.

The systems biology perspective of He et al. (2021) provides important context for interpreting these lipid changes.
They identified lipid metabolism as one of the few metabolic processes exhibiting strong, coordinated regulation at both transcriptional and translational levels, suggesting that membrane remodeling is a priority response that requires tight coupling between gene expression and protein production.
Their identification of UDP-sulfoquinovose synthase (involved in sulfolipid biosynthesis) as part of the core conserved response across maize genotypes, with upregulation at both mRNA and protein levels, validates that the galactolipid and sulfolipid pathways we observe are not artifacts of our specific experimental conditions but rather represent fundamental adaptive mechanisms.
The age-dependency we document represents an additional layer of regulation superimposed on this conserved core response.

The integration of our transcriptomic and lipidomic datasets reveals coordinated regulation of lipid remodeling at multiple enzymatic steps, directly validating Wang et al.'s (2020) proposed metabolic pathway.
Their schematic diagram (their Figure 8) depicts how degradation of phospholipids from the eukaryotic pathway provides phosphatidic acid (PA) and diacylglycerol (DAG) for MGDG synthesis, which then serves as substrate for DGDG production.
Our data support this model at the molecular level: the upregulation of \textit{mgd2} combined with suppression of \textit{peamt2} creates a metabolic bottleneck that redirects lipid flux away from PC biosynthesis and toward galactolipid production.

Wang et al. emphasized that maize, as an 18:3 plant, relies primarily on the eukaryotic (ER-localized) pathway for galactolipid synthesis, distinguishing it from 16:3 plants like \textit{Arabidopsis} that utilize both prokaryotic and eukaryotic pathways.
This pathway dependence is evident in the specific phospholipid species we observe being depleted—predominantly those containing 18-carbon fatty acids characteristic of ER-derived lipids.
The diacylglycerol released from phospholipid breakdown is transported to the chloroplast outer envelope where MGD2 catalyzes MGDG formation, and the resulting MGDG is then converted to DGDG by DGD enzymes.
Our observation that \textit{mgd2} is the dominant upregulated galactolipid biosynthesis gene, while DGD genes show weaker responses, suggests that MGDG synthesis is the rate-limiting step in this cascade, with subsequent DGDG formation occurring through constitutive or post-transcriptionally regulated DGD activity.

The age-dependent amplification of these responses represents a novel dimension absent from Wang et al.'s analysis.
While they demonstrated that both leaf and root tissues undergo membrane remodeling, with leaves showing more dramatic shifts in galactolipid percentages than roots, they did not examine within-leaf heterogeneity across developmental stages.
Our finding that PC34:2 shows both a main effect of phosphorus deficiency and a significant leaf stage interaction ($\log_2\text{FC}=-0.56$ for the interaction term) reveals that the magnitude of phospholipid depletion scales with leaf age.
This developmental gradient suggests that the eukaryotic pathway for galactolipid synthesis is not uniformly active across the leaf canopy but rather is progressively enhanced in older leaves, where senescence-associated lipid turnover intersects with phosphorus limitation stress to create a compounded membrane remodeling response.

Wang et al. reported that DGDG functions as the major bilayer lipid maintaining chloroplast membrane integrity under phosphorus stress, citing previous work showing that DGDG can substitute for phospholipids in forming stable bilayers.
Our lipidomic data support this functional replacement: the increases in digalactosyldiacylglycerol (DGDG) we detect, combined with dramatic decreases in phosphatidylcholine and phosphatidylethanolamine, indicate a wholesale remodeling of membrane composition where galactolipids increasingly dominate the lipid bilayer, particularly in older leaves.
The preservation of some phosphatidylglycerol (PG) species, despite overall phospholipid depletion, aligns with the known essential structural role of PG in photosystem II, where it cannot be substituted by galactolipids.

\subsection*{Developmental Context Determines Nutrient Stress Outcomes}

The leaf age dependency we observed suggests a fundamental principle of plant stress physiology: nutrient limitation does not impose a single metabolic state but rather interacts with pre-existing developmental programs to produce tissue-specific outcomes.
Younger leaves, which are actively expanding and establishing photosynthetic capacity, appear to maintain core anabolic functions even under phosphorus deficiency, albeit at reduced efficiency.
This is evident in the relatively modest changes in chlorophyll synthesis and photosystem component expression in leaf stage one and two under low phosphorus conditions (Fig.~\ref{fig:leaf_physiology_interaction}~E).
These tissues may be protected by preferential allocation of available phosphorus or by the deployment of phosphorus-sparing metabolic alternatives that allow continued growth.

Older leaves, by contrast, undergo a qualitatively different response.
Natural senescence involves programmed dismantling of photosynthetic machinery, remobilization of nutrients to developing sinks, and eventual cell death.
Phosphorus deficiency appears to accelerate this program rather than simply reducing its activity level.
The expression profiles we observed in older leaves under phosphorus limitation resemble advanced senescence rather than young leaves experiencing moderate stress.

This pattern is consistent with previous observations: He et al.(2021) demonstrated in their proteomic and transcriptomic analysis of maize under phosphorus limitation that the cellular response involves coordinated downregulation of ribosome biogenesis and protein synthesis machinery, representing a phosphorus-saving strategy that reduces demand for phosphate-rich rRNA.
Our data extend these findings by showing that this conservation response is amplified in older leaves, which have lower growth demands and can more readily shift to catabolic metabolism.

Notably, KEGG pathway analysis revealed differential regulation within photosynthesis itself.
Genes involved in the Calvin cycle were primarily downregulated as a main effect of phosphorus treatment, whereas light harvesting complex genes showed enrichment specifically in the leaf $\times$ phosphorus interaction set.
This distinction suggests that phosphorus deficiency imposes a hierarchical shutdown: carbon fixation capacity is uniformly reduced across all leaf ages through downregulation of RuBisCO and RuBisCO activase (\textit{rca3}, $\log_2\text{FC}=-3.40$), consistent with the documented reduction in ATP availability under phosphorus limitation.
However, the thylakoid light-capture machinery experiences an additional, age-dependent decline in older leaves, possibly reflecting a programmed strategy to minimize photodamage from excess excitation energy in tissues that can no longer efficiently utilize absorbed light.

Zhang et al. (2014) reported similar selective effects on light-harvesting proteins versus Calvin cycle enzymes in phosphorus-deficient maize leaves, though their bulk-tissue approach could not resolve the developmental gradient we now document.
This differential regulation pattern aligns with He et al.'s (2021) systems-level observations that different aspects of photosynthesis respond through distinct regulatory mechanisms.
They found that photophosphorylation and Calvin cycle proteins cluster into separate co-expression modules, with the former showing stronger responses to phosphorus limitation in shoots than roots.
Our finding that Calvin cycle downregulation is a main effect while light-harvesting downregulation is age-dependent suggests that these modules are not only spatially separated but also temporally decoupled during the progression of leaf senescence under nutrient stress.
The WGCNA framework they employed, which groups proteins based on coordinated abundance changes rather than assumed functional relationships, provides validation that our pathway-level enrichment patterns reflect genuine biological coordination rather than analytical artifacts.

This interpretation is supported by the global expression indices we calculated for chlorophyll metabolism and senescence markers.
Under phosphorus-sufficient conditions, these indices changed linearly with leaf stage ($R^2 = 0.468$ for chlorophyll synthesis; $R^2 = 0.572$ for photosynthesis; $R^2 = 0.277$ for senescence; all $p < 0.001$, Fig.~\ref{fig:leaf_physiology_interaction}~A), reflecting the orderly progression through the leaf lifespan.
Phosphorus deficiency disrupted this linearity specifically in older leaves, creating a steeper trajectory that compressed what would normally be a gradual decline into a more rapid collapse.
The significant interaction terms for these indices across all four gene sets we examined (chlorophyll degradation, $p < 0.01$; chlorophyll synthesis, $p < 0.05$; photosynthesis, $p < 0.001$; senescence, $p < 0.01$) indicate that this is not a localized effect on a few pathways but rather a systemic reprogramming of cellular priorities.

The bifurcation of response trajectories parallels mechanistic insights from senescence biology.
Chloroplast dismantling during senescence is not chaotic decay but rather an organized process involving specific proteases and regulated autophagy.
The downregulation of light-harvesting proteins without proportional effects on stromal enzymes reflects the sequential nature of chloroplast breakdown, where thylakoid membranes are dismantled before stromal proteins are fully degraded.
The accumulation of triacylglycerols in older leaves under phosphorus deficiency may represent a transient storage form for fatty acids released from membrane lipid catabolism, awaiting export to developing sinks or conversion to other metabolites.
This aligns with the general principle that senescing tissues serve as nutrient sources, with older leaves effectively sacrificed to support younger tissues and reproductive development under resource limitation.

\subsection*{Implications for Whole-Plant Resource Management}

The developmental stratification of phosphorus responses has consequences for understanding how maize manages nutrient stress at the whole-plant level.
The strong correlation between leaf age and response magnitude suggests that the plant employs a form of triage during phosphorus limitation, allowing older leaves to senesce more rapidly in order to sustain critical functions in younger, actively growing tissues and reproductive structures.
This is consistent with the ionomic data showing increased seed-to-stover phosphorus ratios under deficiency ($1.99 \pm 0.13$, $p = 1.25 \times 10^{-19}$), indicating prioritized allocation to reproductive sinks at the expense of vegetative tissue maintenance.

The membrane lipid remodeling we observe represents a multilayered adaptation strategy.
First, the replacement of phospholipids with galactolipids conserves phosphorus for use in nucleic acids and ATP synthesis, critical for maintaining basic cellular functions.
Second, the accumulation of triacylglycerols provides temporary storage for fatty acids released from membrane degradation, preventing their toxic accumulation while awaiting export or conversion.
Third, the maintenance of plastidial phosphatidylglycerol (PG), despite severe depletion of other phospholipids, likely reflects its essential structural role in photosystem II, where it cannot be readily substituted.
This hierarchy of lipid class priorities mirrors findings across diverse plant species and validates the evolutionarily conserved nature of this response.

Critically, our age-resolved lipidomic analysis reveals nuances that were obscured in Wang et al.'s (2020) bulk-tissue measurements.
While they reported overall increases in MGDG and DGDG molar percentages (from ~28\% to ~35\% for total galactolipids in leaves), our data demonstrate that this accumulation is not uniform across the leaf canopy but rather concentrated in older leaves, where the membrane remodeling response is most pronounced.
Similarly, while Wang et al. reported overall decreases in PC, PE, PA, and PI percentages (with PC declining from ~15\% to ~10\% of total lipids), we reveal that the magnitude of phospholipid depletion increases systematically with leaf age.

The dramatic reduction in lysophospholipids we observe (LPE18:2, $\log_2\text{FC}=-2.69$; LPC16:1, $\log_2\text{FC}=-3.50$; LPC18:3, $\log_2\text{FC}=-2.64$) extends Wang et al.'s findings by demonstrating that not only intact phospholipids but also their degradation intermediates are depleted, suggesting highly efficient recycling of phosphorus-containing lipid components rather than simple accumulation of breakdown products.

Wang et al. also reported accumulation of triacylglycerols under phosphorus starvation, though they did not provide detailed molecular species characterization.
Our identification of specific TAG species showing dramatic accumulation (TG50:3, $\log_2\text{FC}=3.36$; TG52:6, $\log_2\text{FC}=2.63$; TG54:9, $\log_2\text{FC}=2.25$; TG56:6, $\log_2\text{FC}=12.71$) extends their observations by revealing which fatty acid combinations are preferentially stored.
The particularly striking accumulation of TG56:6, with a fold-change exceeding 6000-fold, suggests that certain very-long-chain polyunsaturated TAG species may serve specialized roles in temporary lipid storage during membrane remodeling.
These TAGs likely represent transient storage forms for fatty acids released from phospholipid degradation, preventing lipotoxicity while awaiting transport to younger tissues or conversion to energy via $\beta$-oxidation, as Wang et al.(2020) hypothesized in their metabolic model.

The coordinated transcriptional regulation we observe at specific enzymatic steps directly validates Wang et al.'s proposed biochemical pathway.
They postulated that phospholipid degradation provides DAG for MGDG synthesis through a pathway involving phospholipase activity followed by PAP (phosphatidic acid phosphatase) action.
Our detection of \textit{pah1} (Zm00001eb289800) showing leaf $\times$ phosphorus interaction supports this model, as this enzyme catalyzes the dephosphorylation of PA to DAG, the committed substrate for either TAG synthesis or transport to the chloroplast for galactolipid production.
The positive interaction term for \textit{pah1} indicates that this branchpoint is developmentally regulated, with older leaves increasingly channeling DAG toward both pathways under phosphorus stress.

Phospholipase D genes, which generate the PA substrate for PAP enzymes, show developmental interactions consistent with Wang et al.'s model: \textit{pld16} (Zm00001eb339870, phospholipase D16) exhibits a positive leaf $\times$ phosphorus interaction ($\log_2\text{FC}=0.56$), indicating enhanced activity specifically in older leaves under phosphorus deficiency.
This pattern suggests that the initial step of phospholipid hydrolysis is developmentally amplified, with older leaves mounting progressively stronger phospholipid degradation responses that feed into both the galactolipid biosynthetic pathway and the TAG storage pathway.

Wang et al. noted that sulfolipid (SQDG) accumulation contributes to membrane stabilization alongside galactolipids, though SQDG showed smaller fold-changes than MGDG/DGDG in their study.
While we detect strong upregulation of both \textit{sqd1} and \textit{sqd2}, these enzymes serve a dual function: they synthesize both SQDG (under phosphorus-replete conditions) and DGGA (under phosphorus-deficient conditions).
The accumulation of DGGA36:4 in our dataset, coupled with the coordinate upregulation of SQD1 and SQD2, confirms that maize employs this additional phosphorus-sparing lipid strategy that extends beyond the simple phospholipid-to-galactolipid replacement Wang et al. primarily documented.

Importantly, He et al.'s finding that only five genes show robust upregulation at both mRNA and protein levels across all genotypes (including UDP-sulfoquinovose synthase, purple acid phosphatase PAP11, and phenylalanine ammonia lyase) suggests a core conserved phosphorus starvation response that operates independently of genetic background.
Our observation that the age-dependent amplification of phosphorus responses occurs consistently across \invfour and control genotypes indicates that developmental modulation of this core response is similarly conserved.
The bifurcation we observe into light-harvesting shutdown versus senescence induction may represent an evolutionarily ancient strategy for managing resource limitation across the leaf canopy, with the specific genes involved being conserved but their deployment being contingent on both phosphorus availability and developmental state.

He et al.'s identification of a mitogen-activated protein kinase as a genotype-specific regulator of sucrose metabolism under low phosphorus suggests that signaling pathways, rather than metabolic enzymes themselves, may be the locus where developmental and environmental signals are integrated.
Future work examining whether similar kinase-mediated signaling pathways show leaf-age-dependent activity could reveal the molecular mechanisms underlying the developmental stratification we document.

The timing of our leaf sampling at 63 days after planting, between the V10 and V12 developmental stages, captured this allocation strategy during a critical transition period when plants were approaching flowering while still expanding leaf area.
The fact that we observed such strong leaf-age-by-phosphorus interactions at this stage suggests that the developmental stratification of stress responses may be particularly important during this window when both vegetative growth and reproductive commitment are occurring simultaneously.
Older leaves, having already contributed to canopy establishment, become expendable and serve as internal nutrient sources, while younger leaves continue to assimilate carbon to support grain filling.

This interpretation aligns with the phenotypic data showing delayed flowering and reduced biomass accumulation under phosphorus deficiency (Fig.~\ref{fig:phenotypes}).
The three-day delay in both anthesis ($3.60 \pm 0.26$ days, $p = 2.3 \times 10^{-20}$) and silking ($3.42 \pm 0.23$ days, $p = 5.7 \times 10^{-22}$) likely reflects the time required for the plant to accumulate sufficient resources through enhanced remobilization from senescing leaves before committing to reproduction.
The progressive decline in stover biomass from 40 to 121 days after planting, which was most severe at the earliest time points (Fig.~\ref{fig:phenotypes}~E), suggests that the initial response to phosphorus limitation involves reduced leaf expansion rates, followed by accelerated senescence of established leaves as the stress persists.

\subsection*{Limited Modulation by \invfour Despite Leaf Age Effects}

Despite the strong leaf-age dependency of phosphorus responses, we found minimal evidence that the \invfour chromosomal inversion modulates this developmental interaction.
The three genes showing significant genotype-by-phosphorus interactions (\textit{aldh2}, \textit{gras80}, \textit{flz22}) are located near but outside the inversion boundaries and did not show additional interactions with leaf stage.
Similarly, while we detected subtle effects of \invfour on specific monogalactosyldiacylglycerol species (MGDG34:2 and MGDG34:3) that varied with both phosphorus availability and leaf age, these effects were restricted to a small number of lipid classes and did not alter the overall pattern of age-dependent stress responses.

The primary effects of \invfour on flowering time (anthesis: $-1.31 \pm 0.26$ days, $p = 4.6 \times 10^{-6}$; silking: $-0.93 \pm 0.23$ days, $p = 1.3 \times 10^{-4}$) and plant height ($6.41 \pm 1.05$ cm, $p = 7.7 \times 10^{-8}$) seem to operate independently of both phosphorus treatment and leaf developmental stage.
This independence suggests that whatever adaptive contributions \invfour makes to highland fitness, they do not involve fine-tuning the coordination between leaf senescence and nutrient stress responses.
The inversion may establish constitutive differences in developmental timing that influence when leaves are produced and when flowering is initiated, but it does not appear to modify how individual leaves respond to phosphorus limitation as they age.

The one exception was cob diameter, which showed a significant interaction between \invfour genotype and phosphorus availability (Fig.~\ref{fig:phenotypes}~D).
While control lines maintained consistent cob diameter regardless of phosphorus treatment ($0.19 \pm 0.70$ cm, $p= 0.79$), \invfour plants developed substantially thinner cobs under phosphorus deficiency ($-2.81 \pm 0.68$ cm, $p= 1.4 \times 10^{-4}$).
This reproductive trait response may reflect altered resource allocation priorities during the critical period when ear structures are being established, possibly linked to the earlier flowering phenology of \invfour plants.
If \invfour plants are initiating reproductive development earlier relative to their vegetative growth stage, they may be more vulnerable to resource limitation during ear formation.
However, this effect did not extend to other reproductive traits such as seed weight or overall yield components, limiting its explanatory power for \invfour's adaptive role.

\subsection*{Rethinking the Hypothesis: Developmental Timing Over Nutrient Acquisition}

Our findings require a reconsideration of how \invfour contributes to highland adaptation in the context of phosphorus-limiting volcanic soils.
The original hypothesis—that \invfour enhances phosphorus acquisition or utilization efficiency through favorable alleles of genes like \textit{ZmPho1;2a}—is not supported by our data.
The absence of significant genotype-by-phosphorus interactions across thousands of genes involved in phosphate scavenging (\textit{pap19}, \textit{pilncr1}, \textit{ips1}), membrane lipid remodeling (\textit{mgd2}, \textit{gpx1}), and metabolic adjustment indicates that \invfour does not fundamentally alter the phosphorus starvation response machinery.

An alternative interpretation is that \invfour's adaptive value in highland environments comes primarily from coordinating developmental timing with the constrained growing season, rather than from direct enhancement of nutrient stress tolerance.
The earlier flowering we observed in \invfour plants, which occurred regardless of phosphorus availability, aligns with previous reports showing that \invfour-highland accelerates flowering specifically at high elevations.
In highland environments with limited temperature sum accumulation and early season termination, completing the life cycle before the onset of unfavorable conditions may be more critical for fitness than marginal improvements in phosphorus acquisition efficiency.

This developmental timing hypothesis is consistent with the leaf age interaction effects we observed.
If \invfour establishes an accelerated developmental program, the coordination between leaf aging and nutrient stress responses becomes particularly important for ensuring adequate resource availability during the compressed reproductive window.
However, our data suggest that this coordination is achieved through constitutive changes in developmental rate rather than through genotype-specific modulation of how individual leaves respond to phosphorus limitation as they age.
The machinery for integrating leaf senescence with phosphorus stress is conserved across genotypes; what differs is the temporal schedule on which that machinery operates.

\subsection*{Methodological Advances and Limitations}

Our experimental design, which systematically sampled four leaf positions per plant and employed both transcriptomic and lipidomic profiling, was specifically structured to detect developmental interactions with environmental treatments.
This approach proved essential for revealing the leaf age dependency of phosphorus responses, which would have been obscured by bulk tissue sampling or single-leaf-stage analyses.
The statistical framework we employed, treating leaf stage as a continuous numerical variable rather than discrete categories, allowed us to quantify interaction effects as rates of change with increasing leaf age, providing more power to detect systematic trends than traditional categorical comparisons.

However, our use of Near Isogenic Lines in the B73 genetic background, while ideal for isolating \invfour effects from confounding variation, may have limited our ability to detect genotype-specific developmental interactions.
The B73 background represents temperate germplasm adapted to high-input agriculture with regular fertilization, and it may lack complementary genetic variants that interact with \invfour to modulate nutrient stress responses in natural highland populations.
The fact that we conducted the field experiment at Rocksprings, Pennsylvania, rather than in the Mexican highlands also means that we captured phosphorus stress effects in isolation from the cold temperatures, altered day length, and UV exposure that characterize the native environment where \invfour has been selected.

The timing of our leaf sampling at a single developmental stage, while providing detailed within-plant spatial resolution, did not capture potential temporal changes in how leaf age interacts with phosphorus availability.
It is possible that the developmental interactions we observed at the V10-V12 stage differ from those that would be evident at earlier seedling stages or later during grain filling.
Future experiments employing time-course sampling across multiple plant developmental stages, combined with spatial sampling across leaf positions, would provide a more complete picture of how phosphorus stress responses are integrated with both leaf aging and whole-plant ontogeny.

\subsection*{Broader Implications for Understanding Chromosomal Inversions in Adaptation}

Our findings contribute to a growing recognition that chromosomal inversions may contribute to local adaptation through multiple mechanisms that operate at different organizational levels.
The classical model of inversions maintaining co-adapted gene complexes through suppressed recombination predicts that inversions should show strong genotype-by-environment interactions when exposed to the selecting environment.
Our failure to detect such interactions for phosphorus stress, despite strong main effects of both \invfour and phosphorus treatment, suggests that at least for this environmental factor, \invfour does not function primarily through environmentally contingent gene expression changes.

Instead, \invfour appears to establish constitutive phenotypic differences in developmental timing that may provide fitness advantages in highland environments through indirect mechanisms.
This interpretation is consistent with theoretical work suggesting that inversions can facilitate local adaptation by coordinating the expression timing of multiple genes rather than by maintaining specific allelic combinations that respond differently to environmental cues.
The flowering time and height effects we observed, which operated independently of phosphorus availability, support this developmental coordination model.

The small set of genotype-by-phosphorus interaction outliers we detected, particularly those overlapping with previously identified flowering time loci, may represent secondary modulatory effects that fine-tune the magnitude of conserved responses rather than primary adaptive mechanisms.
These subtle effects could accumulate over many genes to produce meaningful fitness differences without requiring large-effect, environment-specific transcriptional reprogramming at individual loci.
The challenge for future work will be to determine whether these small-effect interactions, when summed across the many genes within and linked to \invfour, produce ecologically relevant phenotypic differences under naturalistic conditions.

\subsection*{Conclusion}

Our comprehensive analysis demonstrates that the maize phosphorus starvation response is fundamentally shaped by leaf developmental stage, with older leaves experiencing compounded stress that integrates nutrient limitation with natural senescence programs.
We detected quantitatively two distinct molecular developmental trajectories: a negative interaction pattern corresponding to selective shutdown of light-harvesting photosynthetic machinery, and a positive interaction pattern corresponding to induced senescence and enhanced nutrient remobilization.
Gene ontology and pathway enrichment analyses confirm that these trajectories reflect coordinated regulation of functionally related gene sets rather than stochastic variation in stress sensitivity.

The lipidomic dimension of this response parallels the transcriptomic patterns, with older leaves showing enhanced phospholipid degradation, galactolipid accumulation, and glucuronide lipid (DGGA) synthesis under phosphorus limitation.
These findings align with established principles of membrane remodeling during phosphorus stress but reveal that the magnitude of remodeling depends critically on developmental context.
Comparison with previous studies demonstrates that our age-resolved approach captures heterogeneity in stress responses that would be averaged out in bulk-tissue analyses.

Despite this strong leaf-age dependency, the \invfour chromosomal inversion does not substantially modulate these developmental interactions, indicating that its contribution to highland adaptation operates primarily through constitutive effects on developmental timing rather than through enhanced nutrient stress tolerance.
The machinery for coordinating leaf senescence with phosphorus stress is evolutionarily conserved and operates independently of \invfour genotype; what \invfour appears to control is the temporal schedule on which leaves are produced and senesced, which may be advantageous in highland environments with shortened growing seasons.

These findings highlight the need to consider developmental context when interpreting physiological responses to environmental stress and suggest that the adaptive value of chromosomal inversions may lie more in coordinating developmental schedules with environmental constraints than in maintaining environment-specific gene expression programs.
The bifurcation of phosphorus stress responses into light-harvesting shutdown versus senescence induction represents a fundamental principle of how plants integrate nutrient limitation with developmental programs, with implications for understanding crop responses to nutrient deficiency and for designing breeding strategies that optimize resource use efficiency across leaf canopies.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Materials and methods}

\subsection*{\invfour Near Introgressed Lines, growth conditions, experimental design, and phenotype measurements}

To measure the effects of the \invfour in plant field phenotypes and their phosphorus starvation response transcriptome, we used a highland traditional variety carrying the Highland haplotype of \invfour corresponding to the inverted karyotype.
The accession Michoacán 21 (referred to as Mi21), from the Mexican Cónico group, was obtained from the International Maize and Wheat Improvement Center (CIMMYT). 
In contrast, the reference genome of the temperate inbred B73, the recurrent parent for introgression, carries the lowland haplotype corresponding to the standard non-inverted karyotype at \invfour.
From the cross of Mi21 with B73 one F1 individual was backcrossed to B73 for six generations. We selected lines carrying  \invfour with a diagnostic SNP during each cycle using a cleaved amplified polymorphic sequence (CAPS) marker. 
The marker SNP is PZE04175660223 located at position 4:181637780 in the NAM B73v5 \textit{Zea mays} genome assembly.
Amplification of the polymorphic site was done with the following primer pair: \textit{CTGAGCAGGAGATGATGGCCACTC} and \textit{GGAAAGGACATAAAAGAAAGGTGCA}, and subsequently cleaved by \textit{HinfI}.
Plants were genotyped using the CASP marker for selecting heterozygous plants at BC6S2 after selfing seeds of \invfour and CTRL homozygous individuals were selected for the field trial.

Plants were planted on May 26 2022 at the Russell E. Larson Agricultural Research Farm in Rock Springs, Pennsylvania (40°42’36" N 77°57’0" W, 366 m.a.s.l.) in soil classified as a Hagerstown silt loam (fine, mixed, semiactive, mesic Typic Hapludalf).
Experimental conditions were similar to previously described \cite{strock2018}. 
The experiment had a complete block design with two phosphorus (P) levels. 
Low-P fields (5 ppm Melich-3 Phosphorus) and high-P fields (36 ppm Melich-3 Phosphorus) were divided into smaller blocks. 
Three rows per block were planted with a mean stand count of 8 plants per plot, and the plants from the center row were selected for measurements to avoid border effects. 
Fields received fertilization based on treatment requirements. 
Drip irrigation was provided during dry periods. 
Each genotype was replicated four times within its P treatment.

\subsection*{Phenotype analysis}
For stover mass growth curves, a different plant at each time point 40, 50, 60, and harvest, 121 days after planting (DAP), was collected, dried, and weighed for the same row. 
Stover dry mass data was fitted to a logistic growth model using the R package \textit{Growthcurver} \cite{sprouffske2016}.
Maximum Stover dry weight was estimated to be the maximum over the four-time points and not dry weight at harvest.
Ear measurements were taken for one ear per row at harvest. 
We modeled the individual phenotypes with the \textit{R nlme} function as the response variable in a mixed effects model with spatial structure. For each phenotype $y$ we have: 

\begin{eqnarray}
\label{eq:pheno_model}
y_{ijkr} = \beta_{0} + \beta_{1}\text{P}_i + \beta_{2} \textit{\invfour}_j + \beta_{3}[\textit{\invfour} \times \text{P}]_{ij} + u_k + \varepsilon_{ijkr}
\end{eqnarray}

Where the phenotype observation $y_{ijkr}$ corresponds to the plant $r$ in phosphorus treatment $i$ with genotype $j$ in block $k$. The fixed effects coefficients
$\beta_{0}$ for the overall mean,
$\beta_{1}$ for the effect of phosphorus treatment  $i$,
$\beta_{2}$ for the fixed effect of genotype $j$.
One random effect of the block $k$ $(u_{k}) \sim N(0, \sigma_u^2)$, and the residuals  $(\varepsilon_{ijkr}) \sim N(\mathbf{0}, \sigma_\varepsilon^2 \mathbf{\Sigma})$.
We added a correlation structure of the residuals $\mathbf{\Sigma}$ given by a spherical model \cite{pinheiro2000}:


\begin{eqnarray}
\label{eq:sp_model}
\Sigma_{pq} = \begin{cases}
1 & \text{if } d_{pq} = 0 \\
1 - \frac{3d_{pq}}{2\rho} + \frac{d_{pq}^3}{2\rho^3} & \text{if } 0 < d_{pq} < \rho \\
0 & \text{if } d_{pq} \geq \rho
\end{cases}
\end{eqnarray}

Where $d_{pq}$ is the Euclidean distance between plots $p$ and $q$ in the field (based on row and column coordinates), 
and $\rho$ is the range parameter of the spherical model.
We corrected for multiple hypotheses ($H_0: \beta = 0 $) by reporting \textit{t-tests} for the fixed effects  below \textit{FDR} = 0.05. 



\subsection*{Tissue sampling, RNA extraction, and sequencing}
We sampled the plants at 63 DAP when we estimated them to be between v10 to v12 developmental stages. 
We took tissue from the first leaf with a fully developed collar, or the first below the flag leaf, and from there on every other leaf below for a total of four sampled leaves per plant.
These leaves were numbered sequentially from 1 (most apical) to 4 (most basal).
We used four replicate plants per combination of  P treatment and \invfour genotype for a total of 64 tissue samples. 
We took ten disc samples from the leaf tips with a tissue puncher and immediately froze the tissue in 1.5 mL tubes with two steel beads precooled with liquid nitrogen and kept in dry ice until stored at -80\textdegree C.
We extracted total RNA with the QIAGEN RNAeasy Plant Mini Kit  RNA extraction kit following manufacturer procedures (QIAGEN 74904), and RNA samples were quantified in nanodrop and sent to the NCSU Core Genomics Laboratory for sequencing.
Following QC in Bioanalyzer, Illumina libraries were prepared and sequenced in a lane of Novaseq according to manufacturer recommendations.


\subsection*{Plant genotyping}
We followed \cite{brouard2022} for GATK-based RNAseq genotyping of 15 plant samples represented by 60 leaf libraries. 
Briefly, Illumina short reads were mapped to the NAM5 Zea mays B73 genome \cite{hufford2021} using \textit{STAR} \cite{dobin2013}, then we marked duplicates in the resulting BAM alignments, split reads at intron-exon junctions and recalibrated sequence quality per leaf library.
At this point, we used HaploytypeCallerfor for generating gvcfs per plant identified by field row id ($\sim 4$ libraries per plant). 
We did joint sample genotyping afterward with \textit{genotypeGVCFs}. 
Then we filtered for variant quality ( \textit{window 35, cluster, QD < 2.0, FS > 30.0, SOR > 3.0, MQ < 40.0}) for the genotypes and $50\%$ marker completion for individuals. 
This resulted in 200000 markers with $85\%$ complete data for 13 plants.
Finally, we used TASSEL5 K Nearest Neighbour imputing, producing a matrix of 19668 markers at $99.84\%$ completion. 
Shell scripts are  available at the  \href{https://github.com/sawers-rellan-labs/inv4RNA}{inv4RNA github repository}


\subsection*{Differential gene expression and differential lipid analysis}
We aligned reads to the maize Zm-B73-REFERENCE-NAM-5.0 genome using \textit{kallisto} \cite{bray2016}.
The alternative transcript alignment was turned into counts per gene per MB.
We used \textit{voom} to calculate variance according to gene expression levels and counts were converted to $log_2(\text{CPM})$. 
Lipid analysis followed a similar workflow, where we calculated variance weights with \textit{voom} for each lipid MS-spectra peak area and transformed it to $log_2$ scale.
We made a multivariate multiple regression for gene expression and lipid MS signal separately using \textit{limma} \cite{ritchie2015}. For the log transformed expression/signal $Y_{ijrs}$, from leaf $s$, in plant $r$, under phosphorus treatment $i$, with genotype $j$, we have:

 \begin{eqnarray}
% \label{eq:expression_model}
\begin{aligned}
Y_{ijrs} = {}& \beta_0 + \beta_{1}\text{Row}_l + \beta_{2}\text{Column}_{m} + \beta_3 \text{Leaf}_{s} +\beta_4 \text{P}_{i} \\ 
& + \beta_5 \textit{\invfour}_{j} + \beta_6 [\text{P} \times \textit{\invfour}]_{ij} + \varepsilon_{ijrs}
\end{aligned}
\end{eqnarray}
with residuals:
\begin{eqnarray}
\varepsilon_{ijrs} \sim \mathcal{N} (0,\phi\sigma^2)
\end{eqnarray}
We used the leaf stage ($\text{Leaf}_{s}$) as a numerical variable with $s \in \{1,2,3,4\}$ instead of categorical. This implies that $\beta_3$ represents the rate of change of expression with increasing leaf stage number, while the rest of the coefficients were defined as categorical in the same way as in equation \eqref{eq:pheno_model} and \eqref{eq:sp_model}. 
We adjusted the p-values for the t-tests of the linear model coefficients as false discovery rates and genes whose effect had a $FDR <0.05$ were deemed to be differentially expressed.
For phosphorus treatment ($\beta_4$) and \invfour genotype ($\beta_5$) we considered genes with an effect of  $|log_2(\text{Fold Change})| >2$ as top DEGs. 
In the case of the leaf effect, a gene was considered top DEG if $|log_2(\text{Fold Change})|>0.7$, i.e. $>2.1$ between leaf stage 1 and leaf stage 4.
R scripts and expression data are available at the  \href{https://github.com/sawers-rellan-labs/\invfourRNA}{\textit{\invfourRNA} github repository}. 


\subsection*{Gene Ontology and KEGG overrepresentation analysis}

Once we had sets of differentially expressed genes for the three predictors (leaf, -P, \invfour) and two types of gene expression response (upregulated and downregulated), we proceeded to annotate them with gene ontology terms and KEGG pathways using \textit{ClusterProfiler} \cite{yu2012, zicola2024}.  
We started with the B73 NAM v5 gene ontology annotation from \cite{fattel2024} and added GO terms for each intermediate node in the gene ontology tree using the \textit{ClusterProfiler} function \textit{buildGOmap}. 
Then we conducted gene over-representation analysis with the function \textit{compareCluster}, using as universe/background the set of 24011 genes detected in at least one good quality leaf RNAseq library. 
This function calculates the hypergeometric test for overrepresented ontology terms in the specified gene set and returns raw, and FDR-adjusted p-values.
We then manually reviewed the combined 1700  significant (\textit{FDR} $<0.05$) overrepresented GO term associations for the 6 predictor/regulation combinations, and we selected for illustration an \textit{ad hoc} subset with low semantic redundancy.
Similarly, We tested for KEGG pathways over representation using the \textit{enrichKEGG} function from \textit{compareCluster}, which makes the same hypothesis tests on the NCBI REFseq annotation of the B73 NAM assembly. 
Both types of overrepresentation analysis were plotted with the package \textit{DOSE} \cite{yu2015}.


\subsection*{Filtering of \invfour DEGs by phenotype association}

As our data showed evidence of \invfour accelerating flowering time and increasing plant height, we put together a list of candidate genes associated with these two phenotypes to tease out which DEGs were likely contributors to the observed \invfour effect in these traits.
For flowering time, we started with the list of 991 genes compiled by \cite{wang2021} and  62 genes from \cite{li2023a}. 
Then we downloaded the maize data from the GWAS atlas \cite{liu2023} (\textit{gwas\_association\_result\_for\_maize.txt.gz}) and selected genes that overlapped association SNPs for the \href{https://ngdc.cncb.ac.cn/gwas/browse/ontology}{Plant Phenotype and Trait Ontology} term ``days to flowering trait" \textit{PPTO:0000155}.
For this and the following candidate gene list, we considered that a gene overlapped an association SNP if the SNP was located within the 5 kb extended range of the gene model, i.e. as described in the gff gene annotation $\pm 5$ kb.
The final source of associations for flowering time was the phenotypic plasticity study in \cite{tibbs-cortes2024}  from which we used 281 genes with significant GWAS SNPs in the columns \textit{DTS\_slope}, \textit{DTS\_intcp}, \textit{DTA\_slope}, \textit{DTA\_intcp}.
%, with the most significant SNP in this range reported in Table \ref{tab:FT_PH_candidates}.
For plant height, 27 genes from \cite{liu2023}, 1210 genes with GWAS Atlas associations for the term ``plant height" \textit{PPTO:0000126}; and  39 genes overlapping phenotypic plasticity association SNPs for \textit{PH\_slope} and \textit{PH\_intcp} \cite{tibbs-cortes2024}.
The final nonredundant list consisted of a total of 2224 candidate genes for flowering time and 1272 candidates for plant height.

% The genes responding to the introgression of \invfour into the B73 background can be divided into 3 mutually exclusive groups according to their genomic location. 
% First, there are genes that are in the inversion proper; second, those that are in the flanking regions on each side of the introgression and third, those that are located outside the introgression, as described in Table \ref{tab:DEGs_distro}.
% We are introducing a genetic perturbation in the B73 background by introgressing \invfour. 
% However, after 8 generations of selecting for the inversion there still remains 24 Mb of flanking introgression in NILs. 
% This flanking introgression spans 183 genes are are differentially expressed in the \invfour lines with respect to the control plants.

% The change in expression gene outside \invfour is a response to introgression of \invfour and flanking regions because we have experimentally swapped the genotype of the introgressed genes from the B73 reference to the Mi21 allele.

\section{Acknowledgments}
We acknowledge the support of our coffee maker that made this work possible

\bibliography{Inv4mPhosphorus}

\pagebreak

\onecolumn

\section*{Supplement}
\begin{figure}[!hb]
\centering
\includegraphics[width=\textwidth]{figs/biomass.png} % Assuming this is the filename
\caption{
\textbf{Maize Stover Dry Weight Growth Curves and Derived Parameters Highlight Phosphorus-Dependent Effects with No Genotype-by-Environment Interaction.}
(A) Derived logistic growth parameters for CTRL and \textit{Inv4m} genotypes under +P and -P. 
Phosphorus deficiency significantly reduced the Area Under the Curve (AUC) for empirical data (B) and logistic model (C), prolonged the time to reach half maximum stover weight (T$_{1/2}$) (D), and decreased the maximum stover weight (STW$_{\text{max}}$) (E).
Crucially, no significant genotype-by-phosphorus interactions were observed for any of these growth parameters, indicating that \textit{Inv4m} did not modulate the plant's response to phosphorus availability.
Furthermore, there were no significant main effects of the \textit{Inv4m} genotype on these stover grotwh parameters.
\textit{FDR} adjusted \textit{t-test} significance: \textit{n.s.} not significant, $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****).
}
\label{fig:biomass} % Label for your figure
\end{figure}

\pagebreak

\begin{figure*}[!ht]
\centering
\includegraphics[width=\linewidth]{figs/ion_supp.png}
\caption{\textbf{Secondary Ionomic responses of \textit{Inv4m} and control maize lines under phosphorus sufficiency (+P) and deficiency (-P).}
Boxplots show element concentrations (A) in Magnesium (Mg), Manganese (Mn), Potassium (K), and Iron (Fe) in stover and seeds, and Seed/stover ratios (B) for the same four minerals.
Phosphorus deficiency ($-P$) caused a significant reduction in seed Mg (A) and seed Mn (B). No significant effects of either phosphorus treatment or the \textit{Inv4m} genotype were detected for K, Fe, or the seed/stover partition ratios (B) for any of the four elements.
\textit{t-test FDR} adjusted significance: $p < 0.05$ (*), $p < 0.01$ (**), $p < 0.001$ (***), $p < 0.0001$ (****). 
Effect sizes and exact \textit{p values} are reported in Table.}
\label{fig:ionome_supp}
\end{figure*}


\pagebreak


\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes under Phosphorus Deficiency ($\text{-P}$) with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{ccp{7.5cm}cc} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{c}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{l}{\textit{\textbf{Upregulated Genes}}} \\
\hline
Zm00001eb003820 & pilncr1 & pi-deficiency-induced long non-coding RNA1 & 9.0 & 7.70\\
Zm00001eb148590 & ips1 & induced by phosphate starvation1 & 9.0 & 7.10\\
Zm00001eb241920 & gpx1 & glycerophosphodiester phosphodiesterase1 & 9.0 & 6.84\\
Zm00001eb064450 & pap2 & purple acid phosphatase2 & 9.0 & 4.64\\
Zm00001eb154650 & ppa & Inorganic pyrophosphatase 1 & 9.0 & 3.06\\
Zm00001eb280120 & pfk1 & phosphofructose kinase1 & 9.0 & 2.58\\
Zm00001eb063230 & plc6 & phospholipase C6 & 9.0 & 1.90\\
Zm00001eb313760 & flz & FLZ-type domain-containing protein & 8.9 & 3.03\\
Zm00001eb370610 & rfk1 & Riboflavin kinase & 8.9 & 3.98\\
Zm00001eb007180 & gmp & Mannose-1-phosphate guanyltransferase alpha & 8.8 & 2.29\\
Zm00001eb010130 & pap19 & purple acid phosphatase19 & 8.8 & 6.09\\
Zm00001eb099420 & gmps1 & GMP synthase & 4.3 & 9.92\\
Zm00001eb019570 & spx7 & SPX domain-containing membrane protein7 & 4.1 & 8.04\\
Zm00001eb425050 & mdr1 & putative multidrug resistance protein & 3.6 & 8.23\\
Zm00001eb108800 & uam1 & UDP-arabinopyranose mutase & 3.1 & 8.72\\
Zm00001eb034810 & mgd2 & Monogalactosyldiacylglycerol synthase & 2.9 & 11.12\\
Zm00001eb388800 & ltsr1 & Low temperature and salt responsive protein & 2.3 & 9.54\\
\hline
\multicolumn{5}{l}{\textit{\textbf{Downregulated Genes}}} \\
\hline
Zm00001eb433900 & alla1 & allantoinase1 & 6.4 & -1.93\\
Zm00001eb211170 & toc & Translocase of chloroplast, chloroplastic & 5.9 & -1.61\\
Zm00001eb214780 & ccp19 & cysteine protease19 & 5.9 & -1.95\\
Zm00001eb070520 & bhlh148 & bHLH-transcription factor 148 & 5.8 & -2.12\\
Zm00001eb243180 & sdc & Serine decarboxylase & 5.8 & -1.74\\
Zm00001eb377880 & - & - & 5.3 & -1.63\\
Zm00001eb114780 & cfm3 & CRM family member3 & 4.9 & -1.54\\
Zm00001eb405630 & c3h & C3H transcription factor (Fragment) & 4.9 & -1.57\\
Zm00001eb377890 & snf12 & SWI/SNF complex component SNF12-like protein & 4.8 & -1.64\\
Zm00001eb248820 & - & - & 4.7 & -1.84\\
Zm00001eb294690 & peamt2 & phosphoethanolamine N-methyltransferase 2 & 3.8 & -7.17\\
Zm00001eb017120 & tps8 & terpene synthase8 & 3.3 & -4.87\\
Zm00001eb066620 & tut7 & Terminal uridylyltransferase 7 & 2.7 & -4.38\\
Zm00001eb279680 & aaap48 & amino acid/auxin permease48 & 2.3 & -4.39\\
Zm00001eb324550 & nactf132 & NAC-transcription factor 132 & 2.2 & -4.33\\
Zm00001eb292550 & sec14 & SEC14 cytosolic factor family protein / phosphoglyceride transfer family protein & 1.9 & -6.43\\
Zm00001eb410750 & - & - & 1.4 & -4.18\\
\hline
\end{tabular}
\end{table}


\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes for Leaf Stage with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{ccp{7.5cm}cc} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{c}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{l}{\textit{\textbf{Upregulated Genes}}} \\
\hline
Zm00001eb297390 & hir3 & hypersensitive induced reaction3 & 7.4 & 0.80\\
Zm00001eb041700 & gt & Glycosyltransferase & 7.3 & 1.09\\
Zm00001eb305330 & cyp6 & cytochrome P450 & 7.3 & 0.90\\
Zm00001eb037440 & bhlh145 & bHLH-transcription factor 145 & 7.0 & 0.89\\
Zm00001eb293310 & dnaj & DNAJ heat shock N-terminal domain-containing protein & 6.7 & 0.64\\
Zm00001eb407630 & salt1 & SalT homolog1 & 6.5 & 2.54\\
Zm00001eb275060 & - & - & 6.1 & 0.73\\
Zm00001eb098650 & trpp2 & trehalose-6-phosphate phosphatase2 & 6.0 & 1.47\\
Zm00001eb370960 & wrky111 & WRKY-transcription factor 111 & 6.0 & 0.60\\
Zm00001eb163980 & sftp & Surfactant protein B containing protein & 6.0 & 0.53\\
Zm00001eb261620 & imo & Indole-2-monooxygenase & 4.0 & 2.04\\
Zm00001eb422900 & - & - & 2.8 & 1.91\\
Zm00001eb104340 & mutl3 & MUTL protein homolog 3 & 2.3 & 1.96\\
Zm00001eb169810 & sc4mol & sphinganine C4-monooxygenase 1 & 2.2 & 1.79\\
Zm00001eb294140 & - & - & 2.1 & 1.90\\
Zm00001eb002760 & cyp78a & Cytochrome P450 family 78 subfamily A polypeptide 8 & 1.8 & 2.45\\
Zm00001eb137930 & dmas & 2'-deoxymugineic-acid 2'-dioxygenase & 1.6 & 1.89\\
Zm00001eb403420 & abc\_trans & ABC-type Co2+ transport system, permease component & 1.6 & 1.81\\
Zm00001eb054710 & chemo & Chemocyanin & 1.5 & 1.89\\\hline
\multicolumn{5}{l}{\textit{\textbf{Downregulated Genes}}} \\
\hline
Zm00001eb152840 & pcf7 & Transcription factor PCF7 & 7.5 & -1.48\\
Zm00001eb151160 & ntf2 & NTF2 domain-containing protein & 7.5 & -1.15\\
Zm00001eb076680 & sgrl1 & Protein STAY-GREEN LIKE, chloroplastic & 7.5 & -0.95\\
Zm00001eb038410 & ucp4 & Mitochondrial uncoupling protein 4 & 7.5 & -0.70\\
Zm00001eb329970 & tyrtr & Tyrosine-specific transport protein & 7.5 & -0.63\\
Zm00001eb182020 & mph1 & protein MAINTENANCE OF PSII UNDER HIGH LIGHT 1 & 7.5 & -0.61\\
Zm00001eb176730 & ndhb1 & photosynthetic NDH subunit of subcomplex B 1, chloroplastic & 7.5 & -0.52\\
Zm00001eb391900 & tic32 & Short-chain dehydrogenase TIC 32, chloroplastic & 7.4 & -0.60\\
Zm00001eb057540 & zmm4 & Zea mays MADS4 & 7.1 & -3.40\\
Zm00001eb154820 & chk & Choline kinase & 7.1 & -0.53\\
Zm00001eb016200 & bhlh1 & BHLH transcription factor & 6.0 & -3.62\\
Zm00001eb364940 & plt29 & Lipid-transfer protein DIR1 & 5.2 & -2.80\\
Zm00001eb214750 & zmm15 & Zea mays MADS-box 15 & 5.1 & -5.04\\
Zm00001eb320160 & alkt1 & Alkyl transferase & 4.9 & -3.82\\
Zm00001eb169010 & ccp18 & cysteine protease18 & 4.0 & -2.76\\
Zm00001eb090330 & aatr1 & amino acid transporter1 & 3.8 & -3.01\\
Zm00001eb421180 & fp3 & Farnesylated protein 3 & 3.8 & -3.23\\
Zm00001eb411680 & glu2 & beta-glucosidase2 & 2.5 & -5.12\\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\centering
\footnotesize % Reduces font size for the table content
\caption{Selected Differentially Expressed Genes in Leaf $\times$ -P interaction, effect per increased Leaf Stage($\text{-P}$) with PANNZER description.}
\label{table::phosphorusDEGs_2}
\begin{tabular}{ccp{7.5cm}cc} % Adjusted width for Name/Description column
\hline
\textbf{ID} & \textbf{Locus label} & \multicolumn{1}{c}{\textbf{Description}} &   \textbf{$-\log_{10}{\textit{FDR}}$} & \textbf{$\log_2{\text{FC}}$}\\
\hline
\multicolumn{5}{l}{\textit{\textbf{Positively Interacting Genes}}} \\
\hline
Zm00001eb157810 & pk & Pyruvate kinase & 5.6 & 1.18\\
Zm00001eb376160 & mrpa3 & multidrug resistance-associated protein3 & 5.4 & 0.64\\
Zm00001eb063230 & plc6 & phospholipase C6 & 4.5 & 0.56\\
Zm00001eb144680 & rns & Ribonuclease T(2) & 4.4 & 0.61\\
Zm00001eb339870 & pld16 & phospholipase D16 & 4.3 & 0.56\\
Zm00001eb393060 & piplc & PI-PLC X domain-containing protein & 4.0 & 1.15\\
Zm00001eb148030 & gmp1 & mannose-1-phosphate guanylyltransferase1 & 3.9 & 0.69\\
Zm00001eb009430 & htm4 & Heptahelical transmembrane protein 4 & 3.9 & 0.63\\
Zm00001eb011050 & bgal & Beta-galactosidase & 3.9 & 0.53\\
Zm00001eb289800 & pah1 & phosphatidate phosphatase 1 & 3.9 & 0.58\\
Zm00001eb263160 & ring & Zinc finger (C3HC4-type RING finger) family protein & 2.9 & 2.16\\
\hline
\multicolumn{5}{l}{\textit{\textbf{Negatively Interacting  Genes}}} \\
\hline
Zm00001eb359280 & tat & Tat pathway signal sequence family protein & 5.6 & -0.56\\
Zm00001eb207130 & cab & Chlorophyll a-b binding protein, chloroplastic & 5.4 & -1.35\\
Zm00001eb389720 & fbpase & D-fructose-1,6-bisphosphate 1-phosphohydrolase & 5.3 & -0.81\\
Zm00001eb070520 & bhlh148 & bHLH-transcription factor 148 & 5.1 & -0.96\\
Zm00001eb212520 & psad1 & photosystem I subunit d1 & 4.6 & -0.62\\
Zm00001eb179680 & cab & Chlorophyll a-b binding protein, chloroplastic & 4.6 & -0.55\\
Zm00001eb111630 & med33a & Mediator of RNA polymerase II transcription subunit 33A & 4.4 & -0.60\\
Zm00001eb362560 & ndho1 & NADH-plastoquinone oxidoreductase1 & 4.4 & -0.58\\
Zm00001eb214780 & ccp19 & cysteine protease19 & 4.2 & -0.82\\
Zm00001eb071770 & mex1 & maltose excess protein1 & 4.0 & -0.59\\
Zm00001eb256120 &  &  & 3.8 & -1.41\\
Zm00001eb235450 & taf2n & TATA-binding protein-associated factor 2N & 3.6 & -2.07\\
Zm00001eb138960 &  &  & 2.1 & -2.11\\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\centering
\footnotesize
\caption{High-confidence differentially abundant lipids across experimental factors, annotated with IUB nomenclature and lipid class.}
\label{table::differential_lipids}
\begin{tabular}{cccc}
\hline
\textbf{Lipid (IUB)} & \textbf{Class} & \textbf{$-\log_{10}(\textit{FDR})$} & \textbf{$\log_2(\text{FC})$}\\
\hline
\multicolumn{4}{l}{\textit{\textbf{Leaf Tissue Position (Leaf)}}} \\
\hline
\multicolumn{4}{l}{\textit{\textbf{Acccumulated Lipids}}} \\
\hline
LPC18:3 & phospholipid & 3.0 & 1.51\\
LPE18:3 & phospholipid & 2.5 & 1.32\\
PC36:6 & phospholipid & 2.5 & 0.77\\
LPE18:2 & phospholipid & 1.5 & 0.56\\
DGGA36:3 & glycolipid & 1.4 & 0.67\\
\hline
\multicolumn{4}{l}
{\textit{\textbf{Depleted Lipids}}} \\
\hline
DG36:4 & neutral & 2.5 & -0.76\\
DGDG34:1 & glycolipid & 1.9 & -0.56\\
DG26:0 & neutral & 1.7 & -0.67\\
PC36:1 & phospholipid & 1.6 & -0.67\\
DGGA36:4 & glycolipid & 1.5 & -0.72\\
DGDG36:1 & glycolipid & 1.3 & -5.09\\
\hline
\multicolumn{4}{l}{\textit{\textbf{Phosphorus Deficiency (-P)}}} \\
\hline
\multicolumn{4}{l}{\textit{\textbf{Acccumulated Lipids}}} \\
\hline
DGGA36:4 & glycolipid & 1.8 & 1.57\\
TG50:3 & neutral & 1.8 & 3.36\\
TG54:9 & neutral & 1.7 & 2.25\\
TG50:2 & neutral & 1.7 & 2.77\\
TG52:6 & neutral & 1.7 & 2.63\\
TG56:6 & neutral & 1.7 & 12.71\\
TG52:3 & neutral & 1.4 & 2.39\\
\hline
\multicolumn{4}{l}{\textit{\textbf{Depleted Lipids}}} \\
\hline
PC34:2 & phospholipid & 4.8 & -1.60\\
LPE18:2 & phospholipid & 4.1 & -2.69\\
LPC16:1 & phospholipid & 3.2 & -3.50\\
PC32:2 & phospholipid & 3.2 & -2.58\\
PG32:0 & phospholipid & 3.1 & -1.61\\
PE34:4 & phospholipid & 2.2 & -2.06\\
DG26:0 & neutral & 2.1 & -1.82\\
LPC18:3 & phospholipid & 2.1 & -2.64\\
PC32:0 & phospholipid & 1.8 & -2.35\\
PC38:6 & phospholipid & 1.7 & -2.78\\
PE34:3 & phospholipid & 1.7 & -2.20\\
LPE18:3 & phospholipid & 1.7 & -2.02\\
LPC18:2 & phospholipid & 1.6 & -3.29\\
LPE16:0 & phospholipid & 1.4 & -1.86\\
PG34:3 & phospholipid & 1.4 & -3.83\\
PE32:1 & phospholipid & 1.3 & -1.67\\
\hline
\end{tabular}
\end{table}


\end{document}


